==30919== Memcheck, a memory error detector
==30919== Copyright (C) 2002-2011, and GNU GPL'd, by Julian Seward et al.
==30919== Using Valgrind-3.7.0 and LibVEX; rerun with -h for copyright info
==30919== Command: claws-mail
==30919== Parent PID: 30876
==30919== 
==30919== Invalid read of size 8
==30919==    at 0x89AAA0F: wcslen (wcslen.S:48)
==30919==    by 0x89B4D99: wcscoll_l (strcoll_l.c:509)
==30919==    by 0x80BDE79: g_utf8_collate (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.3400.1)
==30919==    by 0x66038F: mgu_list_test_unq_nc (mgutils.c:383)
==30919==    by 0x63D33A: ldapctl_criteria_list_add (ldapctrl.c:344)
==30919==    by 0x611855: addrindex_parse_ldap_attrlist (addrindex.c:1331)
==30919==    by 0x611CD2: addrindex_parse_ldap (addrindex.c:1443)
==30919==    by 0x612371: addrindex_read_index (addrindex.c:1612)
==30919==    by 0x6126A6: addrindex_read_file (addrindex.c:1684)
==30919==    by 0x613DB5: addrindex_read_data (addrindex.c:2307)
==30919==    by 0x609995: addressbook_read_file (addressbook.c:4170)
==30919==    by 0x4D8CDA: main (main.c:1265)
==30919==  Address 0x1192ea48 is 0 bytes after a block of size 40 alloc'd
==30919==    at 0x4C2B3F8: malloc (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==30919==    by 0x80986E0: g_malloc (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.3400.1)
==30919==    by 0x80BEB20: ??? (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.3400.1)
==30919==    by 0x80BDE54: g_utf8_collate (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.3400.1)
==30919==    by 0x66038F: mgu_list_test_unq_nc (mgutils.c:383)
==30919==    by 0x63D33A: ldapctl_criteria_list_add (ldapctrl.c:344)
==30919==    by 0x611855: addrindex_parse_ldap_attrlist (addrindex.c:1331)
==30919==    by 0x611CD2: addrindex_parse_ldap (addrindex.c:1443)
==30919==    by 0x612371: addrindex_read_index (addrindex.c:1612)
==30919==    by 0x6126A6: addrindex_read_file (addrindex.c:1684)
==30919==    by 0x613DB5: addrindex_read_data (addrindex.c:2307)
==30919==    by 0x609995: addressbook_read_file (addressbook.c:4170)
==30919== 
==30919== Invalid read of size 8
==30919==    at 0x89AAA0F: wcslen (wcslen.S:48)
==30919==    by 0x89B4DA8: wcscoll_l (strcoll_l.c:510)
==30919==    by 0x80BDE79: g_utf8_collate (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.3400.1)
==30919==    by 0x66038F: mgu_list_test_unq_nc (mgutils.c:383)
==30919==    by 0x63D33A: ldapctl_criteria_list_add (ldapctrl.c:344)
==30919==    by 0x611855: addrindex_parse_ldap_attrlist (addrindex.c:1331)
==30919==    by 0x611CD2: addrindex_parse_ldap (addrindex.c:1443)
==30919==    by 0x612371: addrindex_read_index (addrindex.c:1612)
==30919==    by 0x6126A6: addrindex_read_file (addrindex.c:1684)
==30919==    by 0x613DB5: addrindex_read_data (addrindex.c:2307)
==30919==    by 0x609995: addressbook_read_file (addressbook.c:4170)
==30919==    by 0x4D8CDA: main (main.c:1265)
==30919==  Address 0x1192edc8 is 0 bytes after a block of size 40 alloc'd
==30919==    at 0x4C2B3F8: malloc (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==30919==    by 0x80986E0: g_malloc (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.3400.1)
==30919==    by 0x80BEB20: ??? (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.3400.1)
==30919==    by 0x80BDE6B: g_utf8_collate (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.3400.1)
==30919==    by 0x66038F: mgu_list_test_unq_nc (mgutils.c:383)
==30919==    by 0x63D33A: ldapctl_criteria_list_add (ldapctrl.c:344)
==30919==    by 0x611855: addrindex_parse_ldap_attrlist (addrindex.c:1331)
==30919==    by 0x611CD2: addrindex_parse_ldap (addrindex.c:1443)
==30919==    by 0x612371: addrindex_read_index (addrindex.c:1612)
==30919==    by 0x6126A6: addrindex_read_file (addrindex.c:1684)
==30919==    by 0x613DB5: addrindex_read_data (addrindex.c:2307)
==30919==    by 0x609995: addressbook_read_file (addressbook.c:4170)
==30919== 
==30919== Invalid read of size 8
==30919==    at 0x89AAA27: wcslen (wcslen.S:55)
==30919==    by 0x89B4D99: wcscoll_l (strcoll_l.c:509)
==30919==    by 0x80BDE79: g_utf8_collate (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.3400.1)
==30919==    by 0x5DCFC2: toolbar_ret_val_from_text (toolbar.c:276)
==30919==    by 0x5DD50E: toolbar_parse_item (toolbar.c:372)
==30919==    by 0x5DE331: toolbar_read_config_file (toolbar.c:750)
==30919==    by 0x5E1ABD: toolbar_create (toolbar.c:1915)
==30919==    by 0x4E58AA: main_window_create (mainwindow.c:1899)
==30919==    by 0x4D8D20: main (main.c:1286)
==30919==  Address 0x135afe88 is 0 bytes after a block of size 56 alloc'd
==30919==    at 0x4C2B3F8: malloc (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==30919==    by 0x80986E0: g_malloc (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.3400.1)
==30919==    by 0x80BEB20: ??? (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.3400.1)
==30919==    by 0x80BDE54: g_utf8_collate (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.3400.1)
==30919==    by 0x5DCFC2: toolbar_ret_val_from_text (toolbar.c:276)
==30919==    by 0x5DD50E: toolbar_parse_item (toolbar.c:372)
==30919==    by 0x5DE331: toolbar_read_config_file (toolbar.c:750)
==30919==    by 0x5E1ABD: toolbar_create (toolbar.c:1915)
==30919==    by 0x4E58AA: main_window_create (mainwindow.c:1899)
==30919==    by 0x4D8D20: main (main.c:1286)
==30919== 
==30919== Invalid read of size 8
==30919==    at 0x89AAA27: wcslen (wcslen.S:55)
==30919==    by 0x89B4DA8: wcscoll_l (strcoll_l.c:510)
==30919==    by 0x80BDE79: g_utf8_collate (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.3400.1)
==30919==    by 0x5DCFC2: toolbar_ret_val_from_text (toolbar.c:276)
==30919==    by 0x5DD50E: toolbar_parse_item (toolbar.c:372)
==30919==    by 0x5DE331: toolbar_read_config_file (toolbar.c:750)
==30919==    by 0x5E1ABD: toolbar_create (toolbar.c:1915)
==30919==    by 0x4E58AA: main_window_create (mainwindow.c:1899)
==30919==    by 0x4D8D20: main (main.c:1286)
==30919==  Address 0x135aff08 is 0 bytes after a block of size 56 alloc'd
==30919==    at 0x4C2B3F8: malloc (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==30919==    by 0x80986E0: g_malloc (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.3400.1)
==30919==    by 0x80BEB20: ??? (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.3400.1)
==30919==    by 0x80BDE6B: g_utf8_collate (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.3400.1)
==30919==    by 0x5DCFC2: toolbar_ret_val_from_text (toolbar.c:276)
==30919==    by 0x5DD50E: toolbar_parse_item (toolbar.c:372)
==30919==    by 0x5DE331: toolbar_read_config_file (toolbar.c:750)
==30919==    by 0x5E1ABD: toolbar_create (toolbar.c:1915)
==30919==    by 0x4E58AA: main_window_create (mainwindow.c:1899)
==30919==    by 0x4D8D20: main (main.c:1286)
==30919== 
==30919== Invalid read of size 8
==30919==    at 0x89AAA0F: wcslen (wcslen.S:48)
==30919==    by 0x89B4DA8: wcscoll_l (strcoll_l.c:510)
==30919==    by 0x80BDE79: g_utf8_collate (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.3400.1)
==30919==    by 0x5DCFC2: toolbar_ret_val_from_text (toolbar.c:276)
==30919==    by 0x5DD50E: toolbar_parse_item (toolbar.c:372)
==30919==    by 0x5DE331: toolbar_read_config_file (toolbar.c:750)
==30919==    by 0x5E1ABD: toolbar_create (toolbar.c:1915)
==30919==    by 0x4E58AA: main_window_create (mainwindow.c:1899)
==30919==    by 0x4D8D20: main (main.c:1286)
==30919==  Address 0x135b6288 is 0 bytes after a block of size 40 alloc'd
==30919==    at 0x4C2B3F8: malloc (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==30919==    by 0x80986E0: g_malloc (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.3400.1)
==30919==    by 0x80BEB20: ??? (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.3400.1)
==30919==    by 0x80BDE6B: g_utf8_collate (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.3400.1)
==30919==    by 0x5DCFC2: toolbar_ret_val_from_text (toolbar.c:276)
==30919==    by 0x5DD50E: toolbar_parse_item (toolbar.c:372)
==30919==    by 0x5DE331: toolbar_read_config_file (toolbar.c:750)
==30919==    by 0x5E1ABD: toolbar_create (toolbar.c:1915)
==30919==    by 0x4E58AA: main_window_create (mainwindow.c:1899)
==30919==    by 0x4D8D20: main (main.c:1286)
==30919== 
==30919== Invalid read of size 8
==30919==    at 0x89AAA0F: wcslen (wcslen.S:48)
==30919==    by 0x89B4D99: wcscoll_l (strcoll_l.c:509)
==30919==    by 0x80BDE79: g_utf8_collate (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.3400.1)
==30919==    by 0x5DCFC2: toolbar_ret_val_from_text (toolbar.c:276)
==30919==    by 0x5DD50E: toolbar_parse_item (toolbar.c:372)
==30919==    by 0x5DE331: toolbar_read_config_file (toolbar.c:750)
==30919==    by 0x5E1ABD: toolbar_create (toolbar.c:1915)
==30919==    by 0x4E58AA: main_window_create (mainwindow.c:1899)
==30919==    by 0x4D8D20: main (main.c:1286)
==30919==  Address 0x135b6978 is 40 bytes inside a block of size 44 alloc'd
==30919==    at 0x4C2B3F8: malloc (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==30919==    by 0x80986E0: g_malloc (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.3400.1)
==30919==    by 0x80BEB20: ??? (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.3400.1)
==30919==    by 0x80BDE54: g_utf8_collate (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.3400.1)
==30919==    by 0x5DCFC2: toolbar_ret_val_from_text (toolbar.c:276)
==30919==    by 0x5DD50E: toolbar_parse_item (toolbar.c:372)
==30919==    by 0x5DE331: toolbar_read_config_file (toolbar.c:750)
==30919==    by 0x5E1ABD: toolbar_create (toolbar.c:1915)
==30919==    by 0x4E58AA: main_window_create (mainwindow.c:1899)
==30919==    by 0x4D8D20: main (main.c:1286)
==30919== 
==30919== Invalid read of size 8
==30919==    at 0x89AAA0F: wcslen (wcslen.S:48)
==30919==    by 0x89B4D99: wcscoll_l (strcoll_l.c:509)
==30919==    by 0x80BDE79: g_utf8_collate (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.3400.1)
==30919==    by 0x5CB0C1: summary_tag_cmp_list (summaryview.c:6190)
==30919==    by 0x80AE419: ??? (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.3400.1)
==30919==    by 0x80AE3EB: ??? (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.3400.1)
==30919==    by 0x5CB104: summary_tags_menu_create (summaryview.c:6204)
==30919==    by 0x5B8A78: summary_init (summaryview.c:997)
==30919==    by 0x4E6B00: main_window_create (mainwindow.c:2180)
==30919==    by 0x4D8D20: main (main.c:1286)
==30919==  Address 0x152f2d30 is 32 bytes inside a block of size 36 alloc'd
==30919==    at 0x4C2B3F8: malloc (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==30919==    by 0x80986E0: g_malloc (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.3400.1)
==30919==    by 0x80BEB20: ??? (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.3400.1)
==30919==    by 0x80BDE54: g_utf8_collate (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.3400.1)
==30919==    by 0x5CB0C1: summary_tag_cmp_list (summaryview.c:6190)
==30919==    by 0x80AE419: ??? (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.3400.1)
==30919==    by 0x80AE3EB: ??? (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.3400.1)
==30919==    by 0x5CB104: summary_tags_menu_create (summaryview.c:6204)
==30919==    by 0x5B8A78: summary_init (summaryview.c:997)
==30919==    by 0x4E6B00: main_window_create (mainwindow.c:2180)
==30919==    by 0x4D8D20: main (main.c:1286)
==30919== 
==30919== Invalid read of size 8
==30919==    at 0x89AAA0F: wcslen (wcslen.S:48)
==30919==    by 0x89B4DA8: wcscoll_l (strcoll_l.c:510)
==30919==    by 0x80BDE79: g_utf8_collate (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.3400.1)
==30919==    by 0x5CB0C1: summary_tag_cmp_list (summaryview.c:6190)
==30919==    by 0x80AE419: ??? (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.3400.1)
==30919==    by 0x5CB104: summary_tags_menu_create (summaryview.c:6204)
==30919==    by 0x5B8A78: summary_init (summaryview.c:997)
==30919==    by 0x4E6B00: main_window_create (mainwindow.c:2180)
==30919==    by 0x4D8D20: main (main.c:1286)
==30919==  Address 0x152f2e60 is 32 bytes inside a block of size 36 alloc'd
==30919==    at 0x4C2B3F8: malloc (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==30919==    by 0x80986E0: g_malloc (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.3400.1)
==30919==    by 0x80BEB20: ??? (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.3400.1)
==30919==    by 0x80BDE6B: g_utf8_collate (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.3400.1)
==30919==    by 0x5CB0C1: summary_tag_cmp_list (summaryview.c:6190)
==30919==    by 0x80AE419: ??? (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.3400.1)
==30919==    by 0x5CB104: summary_tags_menu_create (summaryview.c:6204)
==30919==    by 0x5B8A78: summary_init (summaryview.c:997)
==30919==    by 0x4E6B00: main_window_create (mainwindow.c:2180)
==30919==    by 0x4D8D20: main (main.c:1286)
==30919== 
==30919== Invalid read of size 8
==30919==    at 0x89AAA0F: wcslen (wcslen.S:48)
==30919==    by 0x89B4D99: wcscoll_l (strcoll_l.c:509)
==30919==    by 0x80BDE79: g_utf8_collate (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.3400.1)
==30919==    by 0x4DEFC3: mainwin_tag_cmp_list (mainwindow.c:1246)
==30919==    by 0x80AE419: ??? (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.3400.1)
==30919==    by 0x80AE3EB: ??? (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.3400.1)
==30919==    by 0x4DEFFE: mainwindow_tags_menu_create (mainwindow.c:1258)
==30919==    by 0x4E6C04: main_window_create (mainwindow.c:2202)
==30919==    by 0x4D8D20: main (main.c:1286)
==30919==  Address 0x1537c630 is 32 bytes inside a block of size 36 alloc'd
==30919==    at 0x4C2B3F8: malloc (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==30919==    by 0x80986E0: g_malloc (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.3400.1)
==30919==    by 0x80BEB20: ??? (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.3400.1)
==30919==    by 0x80BDE54: g_utf8_collate (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.3400.1)
==30919==    by 0x4DEFC3: mainwin_tag_cmp_list (mainwindow.c:1246)
==30919==    by 0x80AE419: ??? (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.3400.1)
==30919==    by 0x80AE3EB: ??? (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.3400.1)
==30919==    by 0x4DEFFE: mainwindow_tags_menu_create (mainwindow.c:1258)
==30919==    by 0x4E6C04: main_window_create (mainwindow.c:2202)
==30919==    by 0x4D8D20: main (main.c:1286)
==30919== 
==30919== Invalid read of size 8
==30919==    at 0x89AAA0F: wcslen (wcslen.S:48)
==30919==    by 0x89B4DA8: wcscoll_l (strcoll_l.c:510)
==30919==    by 0x80BDE79: g_utf8_collate (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.3400.1)
==30919==    by 0x4DEFC3: mainwin_tag_cmp_list (mainwindow.c:1246)
==30919==    by 0x80AE419: ??? (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.3400.1)
==30919==    by 0x4DEFFE: mainwindow_tags_menu_create (mainwindow.c:1258)
==30919==    by 0x4E6C04: main_window_create (mainwindow.c:2202)
==30919==    by 0x4D8D20: main (main.c:1286)
==30919==  Address 0x1537c760 is 32 bytes inside a block of size 36 alloc'd
==30919==    at 0x4C2B3F8: malloc (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==30919==    by 0x80986E0: g_malloc (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.3400.1)
==30919==    by 0x80BEB20: ??? (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.3400.1)
==30919==    by 0x80BDE6B: g_utf8_collate (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.3400.1)
==30919==    by 0x4DEFC3: mainwin_tag_cmp_list (mainwindow.c:1246)
==30919==    by 0x80AE419: ??? (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.3400.1)
==30919==    by 0x4DEFFE: mainwindow_tags_menu_create (mainwindow.c:1258)
==30919==    by 0x4E6C04: main_window_create (mainwindow.c:2202)
==30919==    by 0x4D8D20: main (main.c:1286)
==30919== 
==30919== Invalid read of size 8
==30919==    at 0x89AAA27: wcslen (wcslen.S:55)
==30919==    by 0x89B4DA8: wcscoll_l (strcoll_l.c:510)
==30919==    by 0x80BDE79: g_utf8_collate (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.3400.1)
==30919==    by 0x4B7F04: folderview_clist_compare (folderview.c:2498)
==30919==    by 0x6E0180: sink (gtksctree.c:1078)
==30919==    by 0x6E02A2: heap_sort (gtksctree.c:1103)
==30919==    by 0x6E0462: stree_sort (gtksctree.c:1145)
==30919==    by 0x6AD172: gtk_cmctree_post_recursive (gtkcmctree.c:3335)
==30919==    by 0x6AD12F: gtk_cmctree_post_recursive (gtkcmctree.c:3330)
==30919==    by 0x6AD12F: gtk_cmctree_post_recursive (gtkcmctree.c:3330)
==30919==    by 0x6AD12F: gtk_cmctree_post_recursive (gtkcmctree.c:3330)
==30919==    by 0x6E07C7: gtk_sctree_sort_recursive (gtksctree.c:1205)
==30919==  Address 0x1582a090 is 48 bytes inside a block of size 52 alloc'd
==30919==    at 0x4C2B3F8: malloc (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==30919==    by 0x80986E0: g_malloc (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.3400.1)
==30919==    by 0x80BEB20: ??? (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.3400.1)
==30919==    by 0x80BDE6B: g_utf8_collate (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.3400.1)
==30919==    by 0x4B7F04: folderview_clist_compare (folderview.c:2498)
==30919==    by 0x6E0180: sink (gtksctree.c:1078)
==30919==    by 0x6E02A2: heap_sort (gtksctree.c:1103)
==30919==    by 0x6E0462: stree_sort (gtksctree.c:1145)
==30919==    by 0x6AD172: gtk_cmctree_post_recursive (gtkcmctree.c:3335)
==30919==    by 0x6AD12F: gtk_cmctree_post_recursive (gtkcmctree.c:3330)
==30919==    by 0x6AD12F: gtk_cmctree_post_recursive (gtkcmctree.c:3330)
==30919==    by 0x6AD12F: gtk_cmctree_post_recursive (gtkcmctree.c:3330)
==30919== 
==30919== Invalid read of size 8
==30919==    at 0x89AAA0F: wcslen (wcslen.S:48)
==30919==    by 0x89B4D99: wcscoll_l (strcoll_l.c:509)
==30919==    by 0x80BDE79: g_utf8_collate (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.3400.1)
==30919==    by 0x4B7F04: folderview_clist_compare (folderview.c:2498)
==30919==    by 0x6E0180: sink (gtksctree.c:1078)
==30919==    by 0x6E02A2: heap_sort (gtksctree.c:1103)
==30919==    by 0x6E0462: stree_sort (gtksctree.c:1145)
==30919==    by 0x6AD172: gtk_cmctree_post_recursive (gtkcmctree.c:3335)
==30919==    by 0x6AD12F: gtk_cmctree_post_recursive (gtkcmctree.c:3330)
==30919==    by 0x6AD12F: gtk_cmctree_post_recursive (gtkcmctree.c:3330)
==30919==    by 0x6E07C7: gtk_sctree_sort_recursive (gtksctree.c:1205)
==30919==    by 0x4B4F4B: folderview_sort_folders (folderview.c:1715)
==30919==  Address 0x1582a7a0 is 32 bytes inside a block of size 36 alloc'd
==30919==    at 0x4C2B3F8: malloc (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==30919==    by 0x80986E0: g_malloc (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.3400.1)
==30919==    by 0x80BEB20: ??? (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.3400.1)
==30919==    by 0x80BDE54: g_utf8_collate (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.3400.1)
==30919==    by 0x4B7F04: folderview_clist_compare (folderview.c:2498)
==30919==    by 0x6E0180: sink (gtksctree.c:1078)
==30919==    by 0x6E02A2: heap_sort (gtksctree.c:1103)
==30919==    by 0x6E0462: stree_sort (gtksctree.c:1145)
==30919==    by 0x6AD172: gtk_cmctree_post_recursive (gtkcmctree.c:3335)
==30919==    by 0x6AD12F: gtk_cmctree_post_recursive (gtkcmctree.c:3330)
==30919==    by 0x6AD12F: gtk_cmctree_post_recursive (gtkcmctree.c:3330)
==30919==    by 0x6E07C7: gtk_sctree_sort_recursive (gtksctree.c:1205)
==30919== 
==30919== Invalid read of size 8
==30919==    at 0x89AAA0F: wcslen (wcslen.S:48)
==30919==    by 0x89B4DA8: wcscoll_l (strcoll_l.c:510)
==30919==    by 0x80BDE79: g_utf8_collate (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.3400.1)
==30919==    by 0x4B7F04: folderview_clist_compare (folderview.c:2498)
==30919==    by 0x6E01DA: sink (gtksctree.c:1082)
==30919==    by 0x6E02A2: heap_sort (gtksctree.c:1103)
==30919==    by 0x6E0462: stree_sort (gtksctree.c:1145)
==30919==    by 0x6AD172: gtk_cmctree_post_recursive (gtkcmctree.c:3335)
==30919==    by 0x6AD12F: gtk_cmctree_post_recursive (gtkcmctree.c:3330)
==30919==    by 0x6AD12F: gtk_cmctree_post_recursive (gtkcmctree.c:3330)
==30919==    by 0x6E07C7: gtk_sctree_sort_recursive (gtksctree.c:1205)
==30919==    by 0x4B4F4B: folderview_sort_folders (folderview.c:1715)
==30919==  Address 0x1582a8c0 is 32 bytes inside a block of size 36 alloc'd
==30919==    at 0x4C2B3F8: malloc (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==30919==    by 0x80986E0: g_malloc (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.3400.1)
==30919==    by 0x80BEB20: ??? (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.3400.1)
==30919==    by 0x80BDE6B: g_utf8_collate (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.3400.1)
==30919==    by 0x4B7F04: folderview_clist_compare (folderview.c:2498)
==30919==    by 0x6E01DA: sink (gtksctree.c:1082)
==30919==    by 0x6E02A2: heap_sort (gtksctree.c:1103)
==30919==    by 0x6E0462: stree_sort (gtksctree.c:1145)
==30919==    by 0x6AD172: gtk_cmctree_post_recursive (gtkcmctree.c:3335)
==30919==    by 0x6AD12F: gtk_cmctree_post_recursive (gtkcmctree.c:3330)
==30919==    by 0x6AD12F: gtk_cmctree_post_recursive (gtkcmctree.c:3330)
==30919==    by 0x6E07C7: gtk_sctree_sort_recursive (gtksctree.c:1205)
==30919== 
==30919== Invalid read of size 8
==30919==    at 0x89AAA27: wcslen (wcslen.S:55)
==30919==    by 0x89B4D99: wcscoll_l (strcoll_l.c:509)
==30919==    by 0x80BDE79: g_utf8_collate (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.3400.1)
==30919==    by 0x4B7F04: folderview_clist_compare (folderview.c:2498)
==30919==    by 0x6E0180: sink (gtksctree.c:1078)
==30919==    by 0x6E02A2: heap_sort (gtksctree.c:1103)
==30919==    by 0x6E0462: stree_sort (gtksctree.c:1145)
==30919==    by 0x6AD172: gtk_cmctree_post_recursive (gtkcmctree.c:3335)
==30919==    by 0x6AD12F: gtk_cmctree_post_recursive (gtkcmctree.c:3330)
==30919==    by 0x6E07C7: gtk_sctree_sort_recursive (gtksctree.c:1205)
==30919==    by 0x4B4F4B: folderview_sort_folders (folderview.c:1715)
==30919==    by 0x4B50FD: folderview_append_folder (folderview.c:1739)
==30919==  Address 0x1582e748 is 0 bytes after a block of size 56 alloc'd
==30919==    at 0x4C2B3F8: malloc (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==30919==    by 0x80986E0: g_malloc (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.3400.1)
==30919==    by 0x80BEB20: ??? (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.3400.1)
==30919==    by 0x80BDE54: g_utf8_collate (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.3400.1)
==30919==    by 0x4B7F04: folderview_clist_compare (folderview.c:2498)
==30919==    by 0x6E0180: sink (gtksctree.c:1078)
==30919==    by 0x6E02A2: heap_sort (gtksctree.c:1103)
==30919==    by 0x6E0462: stree_sort (gtksctree.c:1145)
==30919==    by 0x6AD172: gtk_cmctree_post_recursive (gtkcmctree.c:3335)
==30919==    by 0x6AD12F: gtk_cmctree_post_recursive (gtkcmctree.c:3330)
==30919==    by 0x6E07C7: gtk_sctree_sort_recursive (gtksctree.c:1205)
==30919==    by 0x4B4F4B: folderview_sort_folders (folderview.c:1715)
==30919== 
==30919== Invalid read of size 8
==30919==    at 0x89AAA3F: wcslen (wcslen.S:62)
==30919==    by 0x89B4D99: wcscoll_l (strcoll_l.c:509)
==30919==    by 0x80BDE79: g_utf8_collate (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.3400.1)
==30919==    by 0x4B7F04: folderview_clist_compare (folderview.c:2498)
==30919==    by 0x6E01DA: sink (gtksctree.c:1082)
==30919==    by 0x6E02A2: heap_sort (gtksctree.c:1103)
==30919==    by 0x6E0462: stree_sort (gtksctree.c:1145)
==30919==    by 0x6AD172: gtk_cmctree_post_recursive (gtkcmctree.c:3335)
==30919==    by 0x6E07C7: gtk_sctree_sort_recursive (gtksctree.c:1205)
==30919==    by 0x4B4F4B: folderview_sort_folders (folderview.c:1715)
==30919==    by 0x4B50FD: folderview_append_folder (folderview.c:1739)
==30919==    by 0x4B2AC2: folderview_set_folders (folderview.c:975)
==30919==  Address 0x15833b88 is 0 bytes after a block of size 72 alloc'd
==30919==    at 0x4C2B3F8: malloc (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==30919==    by 0x80986E0: g_malloc (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.3400.1)
==30919==    by 0x80BEB20: ??? (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.3400.1)
==30919==    by 0x80BDE54: g_utf8_collate (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.3400.1)
==30919==    by 0x4B7F04: folderview_clist_compare (folderview.c:2498)
==30919==    by 0x6E01DA: sink (gtksctree.c:1082)
==30919==    by 0x6E02A2: heap_sort (gtksctree.c:1103)
==30919==    by 0x6E0462: stree_sort (gtksctree.c:1145)
==30919==    by 0x6AD172: gtk_cmctree_post_recursive (gtkcmctree.c:3335)
==30919==    by 0x6E07C7: gtk_sctree_sort_recursive (gtksctree.c:1205)
==30919==    by 0x4B4F4B: folderview_sort_folders (folderview.c:1715)
==30919==    by 0x4B50FD: folderview_append_folder (folderview.c:1739)
==30919== 
==30919== Invalid read of size 8
==30919==    at 0x89AAA3F: wcslen (wcslen.S:62)
==30919==    by 0x89B4DA8: wcscoll_l (strcoll_l.c:510)
==30919==    by 0x80BDE79: g_utf8_collate (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.3400.1)
==30919==    by 0x4B7F04: folderview_clist_compare (folderview.c:2498)
==30919==    by 0x6E0180: sink (gtksctree.c:1078)
==30919==    by 0x6E0320: heap_sort (gtksctree.c:1109)
==30919==    by 0x6E0462: stree_sort (gtksctree.c:1145)
==30919==    by 0x6AD172: gtk_cmctree_post_recursive (gtkcmctree.c:3335)
==30919==    by 0x6E07C7: gtk_sctree_sort_recursive (gtksctree.c:1205)
==30919==    by 0x4B4F4B: folderview_sort_folders (folderview.c:1715)
==30919==    by 0x4B50FD: folderview_append_folder (folderview.c:1739)
==30919==    by 0x4B2AC2: folderview_set_folders (folderview.c:975)
==30919==  Address 0x158355b8 is 0 bytes after a block of size 72 alloc'd
==30919==    at 0x4C2B3F8: malloc (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==30919==    by 0x80986E0: g_malloc (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.3400.1)
==30919==    by 0x80BEB20: ??? (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.3400.1)
==30919==    by 0x80BDE6B: g_utf8_collate (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.3400.1)
==30919==    by 0x4B7F04: folderview_clist_compare (folderview.c:2498)
==30919==    by 0x6E0180: sink (gtksctree.c:1078)
==30919==    by 0x6E0320: heap_sort (gtksctree.c:1109)
==30919==    by 0x6E0462: stree_sort (gtksctree.c:1145)
==30919==    by 0x6AD172: gtk_cmctree_post_recursive (gtkcmctree.c:3335)
==30919==    by 0x6E07C7: gtk_sctree_sort_recursive (gtksctree.c:1205)
==30919==    by 0x4B4F4B: folderview_sort_folders (folderview.c:1715)
==30919==    by 0x4B50FD: folderview_append_folder (folderview.c:1739)
==30919== 
==30919== Invalid read of size 8
==30919==    at 0x89AAA6B: wcslen (wcslen.S:75)
==30919==    by 0x89B4D99: wcscoll_l (strcoll_l.c:509)
==30919==    by 0x80BDE79: g_utf8_collate (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.3400.1)
==30919==    by 0x4B7F04: folderview_clist_compare (folderview.c:2498)
==30919==    by 0x6E01DA: sink (gtksctree.c:1082)
==30919==    by 0x6E02A2: heap_sort (gtksctree.c:1103)
==30919==    by 0x6E0462: stree_sort (gtksctree.c:1145)
==30919==    by 0x6AD172: gtk_cmctree_post_recursive (gtkcmctree.c:3335)
==30919==    by 0x6E07C7: gtk_sctree_sort_recursive (gtksctree.c:1205)
==30919==    by 0x4B4F4B: folderview_sort_folders (folderview.c:1715)
==30919==    by 0x4B50FD: folderview_append_folder (folderview.c:1739)
==30919==    by 0x4B2AC2: folderview_set_folders (folderview.c:975)
==30919==  Address 0x16991eb8 is 0 bytes after a block of size 104 alloc'd
==30919==    at 0x4C2B3F8: malloc (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==30919==    by 0x80986E0: g_malloc (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.3400.1)
==30919==    by 0x80BEB20: ??? (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.3400.1)
==30919==    by 0x80BDE54: g_utf8_collate (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.3400.1)
==30919==    by 0x4B7F04: folderview_clist_compare (folderview.c:2498)
==30919==    by 0x6E01DA: sink (gtksctree.c:1082)
==30919==    by 0x6E02A2: heap_sort (gtksctree.c:1103)
==30919==    by 0x6E0462: stree_sort (gtksctree.c:1145)
==30919==    by 0x6AD172: gtk_cmctree_post_recursive (gtkcmctree.c:3335)
==30919==    by 0x6E07C7: gtk_sctree_sort_recursive (gtksctree.c:1205)
==30919==    by 0x4B4F4B: folderview_sort_folders (folderview.c:1715)
==30919==    by 0x4B50FD: folderview_append_folder (folderview.c:1739)
==30919== 
==30919== Invalid read of size 4
==30919==    at 0x1C6BCF32: PyObject_Free (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C65E5DA: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C680B8E: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C6812EB: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C638612: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C6817FC: PyImport_ImportModuleLevel (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C5D703E: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C6541FD: PyObject_Call (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C654B06: PyEval_CallObjectWithKeywords (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C57F5C2: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C584C37: PyEval_EvalCodeEx (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C584D71: PyEval_EvalCode (in /usr/lib/libpython2.7.so.1.0)
==30919==  Address 0x1cca1020 is 2,000 bytes inside a block of size 2,731 free'd
==30919==    at 0x4C2A739: free (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==30919==    by 0x1C67B36A: PyMarshal_ReadLastObjectFromFile (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C680A0D: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C6812EB: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C638612: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C6817FC: PyImport_ImportModuleLevel (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C5D703E: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C6541FD: PyObject_Call (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C654B06: PyEval_CallObjectWithKeywords (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C57F5C2: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C584C37: PyEval_EvalCodeEx (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C584D71: PyEval_EvalCode (in /usr/lib/libpython2.7.so.1.0)
==30919== 
==30919== Invalid read of size 4
==30919==    at 0x1C6BCF32: PyObject_Free (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C6A37CA: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C65E65A: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C680B8E: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C6812EB: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C638612: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C6817FC: PyImport_ImportModuleLevel (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C5D703E: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C6541FD: PyObject_Call (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C654B06: PyEval_CallObjectWithKeywords (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C57F5C2: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C584C37: PyEval_EvalCodeEx (in /usr/lib/libpython2.7.so.1.0)
==30919==  Address 0x1cca1020 is 2,000 bytes inside a block of size 2,731 free'd
==30919==    at 0x4C2A739: free (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==30919==    by 0x1C67B36A: PyMarshal_ReadLastObjectFromFile (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C680A0D: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C6812EB: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C638612: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C6817FC: PyImport_ImportModuleLevel (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C5D703E: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C6541FD: PyObject_Call (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C654B06: PyEval_CallObjectWithKeywords (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C57F5C2: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C584C37: PyEval_EvalCodeEx (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C584D71: PyEval_EvalCode (in /usr/lib/libpython2.7.so.1.0)
==30919== 
==30919== Invalid read of size 4
==30919==    at 0x1C6BCF32: PyObject_Free (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C6A37CA: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C65E64A: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C680B8E: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C6812EB: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C638612: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C6817FC: PyImport_ImportModuleLevel (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C5D703E: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C6541FD: PyObject_Call (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C654B06: PyEval_CallObjectWithKeywords (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C57F5C2: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C584C37: PyEval_EvalCodeEx (in /usr/lib/libpython2.7.so.1.0)
==30919==  Address 0x1cca1020 is 2,000 bytes inside a block of size 2,731 free'd
==30919==    at 0x4C2A739: free (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==30919==    by 0x1C67B36A: PyMarshal_ReadLastObjectFromFile (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C680A0D: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C6812EB: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C638612: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C6817FC: PyImport_ImportModuleLevel (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C5D703E: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C6541FD: PyObject_Call (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C654B06: PyEval_CallObjectWithKeywords (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C57F5C2: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C584C37: PyEval_EvalCodeEx (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C584D71: PyEval_EvalCode (in /usr/lib/libpython2.7.so.1.0)
==30919== 
==30919== Invalid read of size 4
==30919==    at 0x1C6BCF32: PyObject_Free (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C65E5B0: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C680B8E: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C6812EB: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C638612: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C6817FC: PyImport_ImportModuleLevel (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C5D703E: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C6541FD: PyObject_Call (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C654B06: PyEval_CallObjectWithKeywords (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C57F5C2: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C584C37: PyEval_EvalCodeEx (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C584D71: PyEval_EvalCode (in /usr/lib/libpython2.7.so.1.0)
==30919==  Address 0x1ccd9020 is 2,592 bytes inside a block of size 3,243 free'd
==30919==    at 0x4C2A739: free (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==30919==    by 0x1C67B36A: PyMarshal_ReadLastObjectFromFile (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C680A0D: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C6812EB: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C638612: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C6817FC: PyImport_ImportModuleLevel (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C5D703E: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C6541FD: PyObject_Call (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C654B06: PyEval_CallObjectWithKeywords (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C57F5C2: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C584C37: PyEval_EvalCodeEx (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C584D71: PyEval_EvalCode (in /usr/lib/libpython2.7.so.1.0)
==30919== 
==30919== Conditional jump or move depends on uninitialised value(s)
==30919==    at 0x1C6BCF3B: PyObject_Free (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C65E5DA: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C680B8E: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C6812EB: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C638612: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C6817FC: PyImport_ImportModuleLevel (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C5D703E: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C6541FD: PyObject_Call (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C654B06: PyEval_CallObjectWithKeywords (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C57F5C2: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C584C37: PyEval_EvalCodeEx (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C584D71: PyEval_EvalCode (in /usr/lib/libpython2.7.so.1.0)
==30919== 
==30919== Use of uninitialised value of size 8
==30919==    at 0x1C6BCF53: PyObject_Free (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C65E5DA: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C680B8E: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C6812EB: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C638612: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C6817FC: PyImport_ImportModuleLevel (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C5D703E: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C6541FD: PyObject_Call (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C654B06: PyEval_CallObjectWithKeywords (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C57F5C2: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C584C37: PyEval_EvalCodeEx (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C584D71: PyEval_EvalCode (in /usr/lib/libpython2.7.so.1.0)
==30919== 
==30919== Invalid read of size 4
==30919==    at 0x1C6BCF32: PyObject_Free (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C65E5DA: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C6A37B1: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C65E65A: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C680B8E: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C6812EB: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C638612: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C6817FC: PyImport_ImportModuleLevel (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C5D703E: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C6541FD: PyObject_Call (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C654B06: PyEval_CallObjectWithKeywords (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C57F5C2: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==30919==  Address 0x1cd6c020 is 7,248 bytes inside a block of size 9,383 free'd
==30919==    at 0x4C2A739: free (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==30919==    by 0x1C67B36A: PyMarshal_ReadLastObjectFromFile (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C680A0D: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C6812EB: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C638612: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C6817FC: PyImport_ImportModuleLevel (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C5D703E: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C6541FD: PyObject_Call (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C654B06: PyEval_CallObjectWithKeywords (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C57F5C2: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C584C37: PyEval_EvalCodeEx (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C584D71: PyEval_EvalCode (in /usr/lib/libpython2.7.so.1.0)
==30919== 
==30919== Invalid read of size 4
==30919==    at 0x1C6BCF32: PyObject_Free (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C6A37CA: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C65E65A: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C6A37B1: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C65E65A: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C680B8E: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C6812EB: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C638612: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C6817FC: PyImport_ImportModuleLevel (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C5D703E: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C6541FD: PyObject_Call (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C654B06: PyEval_CallObjectWithKeywords (in /usr/lib/libpython2.7.so.1.0)
==30919==  Address 0x1cd6c020 is 7,248 bytes inside a block of size 9,383 free'd
==30919==    at 0x4C2A739: free (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==30919==    by 0x1C67B36A: PyMarshal_ReadLastObjectFromFile (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C680A0D: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C6812EB: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C638612: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C6817FC: PyImport_ImportModuleLevel (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C5D703E: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C6541FD: PyObject_Call (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C654B06: PyEval_CallObjectWithKeywords (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C57F5C2: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C584C37: PyEval_EvalCodeEx (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C584D71: PyEval_EvalCode (in /usr/lib/libpython2.7.so.1.0)
==30919== 
==30919== Invalid read of size 4
==30919==    at 0x1C6BCF32: PyObject_Free (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C6A37CA: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C65E64A: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C6A37B1: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C65E65A: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C680B8E: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C6812EB: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C638612: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C6817FC: PyImport_ImportModuleLevel (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C5D703E: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C6541FD: PyObject_Call (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C654B06: PyEval_CallObjectWithKeywords (in /usr/lib/libpython2.7.so.1.0)
==30919==  Address 0x1cd6d020 is 48 bytes inside a block of size 368 free'd
==30919==    at 0x4C2B4F0: realloc (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==30919==    by 0x1C607ADD: PyList_Append (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C67AEBB: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C67A194: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C67A750: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C67A194: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C67A73C: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C67A194: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C67A73C: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C67B257: PyMarshal_ReadObjectFromString (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C67B35D: PyMarshal_ReadLastObjectFromFile (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C680A0D: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919== 
==30919== Conditional jump or move depends on uninitialised value(s)
==30919==    at 0x1C6BCF3B: PyObject_Free (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C65E5B0: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C6A37B1: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C65E65A: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C680B8E: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C6812EB: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C638612: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C6817FC: PyImport_ImportModuleLevel (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C5D703E: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C6541FD: PyObject_Call (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C654B06: PyEval_CallObjectWithKeywords (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C57F5C2: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==30919== 
==30919== Use of uninitialised value of size 8
==30919==    at 0x1C6BCF53: PyObject_Free (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C65E5B0: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C6A37B1: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C65E65A: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C680B8E: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C6812EB: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C638612: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C6817FC: PyImport_ImportModuleLevel (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C5D703E: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C6541FD: PyObject_Call (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C654B06: PyEval_CallObjectWithKeywords (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C57F5C2: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==30919== 
==30919== Invalid read of size 4
==30919==    at 0x1C6BCF32: PyObject_Free (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C65E5B0: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C6A37B1: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C65E65A: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C680B8E: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C6812EB: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C638612: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C6817FC: PyImport_ImportModuleLevel (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C5D703E: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C6541FD: PyObject_Call (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C654B06: PyEval_CallObjectWithKeywords (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C57F5C2: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==30919==  Address 0x1cd86020 is 16 bytes before a block of size 40 free'd
==30919==    at 0x4C2B4F0: realloc (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==30919==    by 0x1C607ADD: PyList_Append (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C685766: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C6541FD: PyObject_Call (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C654B06: PyEval_CallObjectWithKeywords (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C68598E: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C6861CF: PyType_Ready (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C591855: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C64A259: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C583D71: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C584C37: PyEval_EvalCodeEx (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C5DACB5: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919== 
==30919== Invalid read of size 4
==30919==    at 0x1C6BCF32: PyObject_Free (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C65E5EA: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C680B8E: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C6812EB: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C638612: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C6817FC: PyImport_ImportModuleLevel (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C5D703E: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C6541FD: PyObject_Call (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C654B06: PyEval_CallObjectWithKeywords (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C57F5C2: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C584C37: PyEval_EvalCodeEx (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C584D71: PyEval_EvalCode (in /usr/lib/libpython2.7.so.1.0)
==30919==  Address 0x1b053020 is 1,248 bytes inside a block of size 1,608 free'd
==30919==    at 0x4C2B4F0: realloc (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==30919==    by 0x1C607ADD: PyList_Append (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C67AEBB: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C67A7E4: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C67B257: PyMarshal_ReadObjectFromString (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C67B35D: PyMarshal_ReadLastObjectFromFile (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C680A0D: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C6812EB: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C638612: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C6817FC: PyImport_ImportModuleLevel (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C5D703E: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C6541FD: PyObject_Call (in /usr/lib/libpython2.7.so.1.0)
==30919== 
==30919== Invalid read of size 4
==30919==    at 0x1C6BCF32: PyObject_Free (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C6A37B1: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C65E65A: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C680B8E: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C6812EB: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C638612: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C6817FC: PyImport_ImportModuleLevel (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C5D703E: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C6541FD: PyObject_Call (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C654B06: PyEval_CallObjectWithKeywords (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C57F5C2: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C584C37: PyEval_EvalCodeEx (in /usr/lib/libpython2.7.so.1.0)
==30919==  Address 0x1ce01020 is 320 bytes inside a block of size 704 free'd
==30919==    at 0x4C2B4F0: realloc (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==30919==    by 0x1C607ADD: PyList_Append (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C67AEBB: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C67A194: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C67A73C: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C67B257: PyMarshal_ReadObjectFromString (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C67B35D: PyMarshal_ReadLastObjectFromFile (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C680A0D: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C6812EB: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C638612: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C6817FC: PyImport_ImportModuleLevel (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C5D703E: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919== 
==30919== Conditional jump or move depends on uninitialised value(s)
==30919==    at 0x1C6BCF3B: PyObject_Free (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C65E5B0: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C680B8E: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C6812EB: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C638612: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C6817FC: PyImport_ImportModuleLevel (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C5D703E: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C6541FD: PyObject_Call (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C654B06: PyEval_CallObjectWithKeywords (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C57F5C2: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C583E1A: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C584C37: PyEval_EvalCodeEx (in /usr/lib/libpython2.7.so.1.0)
==30919== 
==30919== Use of uninitialised value of size 8
==30919==    at 0x1C6BCF53: PyObject_Free (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C65E5B0: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C680B8E: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C6812EB: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C638612: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C6817FC: PyImport_ImportModuleLevel (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C5D703E: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C6541FD: PyObject_Call (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C654B06: PyEval_CallObjectWithKeywords (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C57F5C2: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C583E1A: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C584C37: PyEval_EvalCodeEx (in /usr/lib/libpython2.7.so.1.0)
==30919== 
==30919== Invalid read of size 4
==30919==    at 0x1C6BCF32: PyObject_Free (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C6958EB: PyGrammar_AddAccelerators (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C695BD7: PyParser_New (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C6802D4: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C5F1F27: PyParser_ASTFromString (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C5DAE5C: PyRun_StringFlags (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C580537: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C584C37: PyEval_EvalCodeEx (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C58282D: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C584C37: PyEval_EvalCodeEx (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C58282D: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C583E1A: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==30919==  Address 0x1ceb0020 is 16 bytes before a block of size 41 free'd
==30919==    at 0x4C2B4F0: realloc (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==30919==    by 0x1C5B4602: PyTokenizer_FromString (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C5B488F: PyParser_ParseStringFlagsFilenameEx (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C5F1F27: PyParser_ASTFromString (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C5DAE5C: PyRun_StringFlags (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C580537: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C584C37: PyEval_EvalCodeEx (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C58282D: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C584C37: PyEval_EvalCodeEx (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C58282D: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C583E1A: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C583E1A: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==30919== 
==30919== Invalid read of size 4
==30919==    at 0x1C6BCF32: PyObject_Free (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C6BF74D: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C599FE7: PyAST_Compile (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C5DAE76: PyRun_StringFlags (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C580537: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C584C37: PyEval_EvalCodeEx (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C58282D: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C584C37: PyEval_EvalCodeEx (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C58282D: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C583E1A: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C583E1A: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C584C37: PyEval_EvalCodeEx (in /usr/lib/libpython2.7.so.1.0)
==30919==  Address 0x1cf10020 is not stack'd, malloc'd or (recently) free'd
==30919== 
==30919== Invalid read of size 4
==30919==    at 0x1C6BCF32: PyObject_Free (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C599FE7: PyAST_Compile (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C5DAE76: PyRun_StringFlags (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C580537: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C584C37: PyEval_EvalCodeEx (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C58282D: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C584C37: PyEval_EvalCodeEx (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C58282D: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C583E1A: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C583E1A: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C584C37: PyEval_EvalCodeEx (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C584D71: PyEval_EvalCode (in /usr/lib/libpython2.7.so.1.0)
==30919==  Address 0x1cf10020 is not stack'd, malloc'd or (recently) free'd
==30919== 
==30919== Invalid read of size 4
==30919==    at 0x1C6BCF32: PyObject_Free (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C65E5B0: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C5DAED1: PyRun_StringFlags (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C580537: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C584C37: PyEval_EvalCodeEx (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C58282D: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C584C37: PyEval_EvalCodeEx (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C58282D: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C583E1A: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C583E1A: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C584C37: PyEval_EvalCodeEx (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C584D71: PyEval_EvalCode (in /usr/lib/libpython2.7.so.1.0)
==30919==  Address 0x1cf10020 is not stack'd, malloc'd or (recently) free'd
==30919== 
==30919== Conditional jump or move depends on uninitialised value(s)
==30919==    at 0x1C69540B: PyObject_Realloc (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C695613: PyNode_AddChild (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C620032: PyParser_AddToken (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C680351: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C5F1F27: PyParser_ASTFromString (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C5DAE5C: PyRun_StringFlags (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C580537: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C584C37: PyEval_EvalCodeEx (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C58282D: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C584C37: PyEval_EvalCodeEx (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C58282D: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C583E1A: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==30919== 
==30919== Use of uninitialised value of size 8
==30919==    at 0x1C69541F: PyObject_Realloc (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C695613: PyNode_AddChild (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C620032: PyParser_AddToken (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C680351: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C5F1F27: PyParser_ASTFromString (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C5DAE5C: PyRun_StringFlags (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C580537: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C584C37: PyEval_EvalCodeEx (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C58282D: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C584C37: PyEval_EvalCodeEx (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C58282D: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C583E1A: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==30919== 
==30919== Invalid read of size 4
==30919==    at 0x1C6BCF32: PyObject_Free (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C695FD7: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C695EB6: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C695EB6: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C69607D: PyNode_Free (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C5F1ED7: PyParser_ASTFromString (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C5DAE5C: PyRun_StringFlags (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C580537: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C584C37: PyEval_EvalCodeEx (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C58282D: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C584C37: PyEval_EvalCodeEx (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C58282D: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==30919==  Address 0x1cf1d020 is 33,344 bytes inside a block of size 36,032 free'd
==30919==    at 0x4C2A739: free (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==30919==    by 0x1C680527: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C5F1F27: PyParser_ASTFromString (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C5DAE5C: PyRun_StringFlags (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C580537: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C584C37: PyEval_EvalCodeEx (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C58282D: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C584C37: PyEval_EvalCodeEx (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C58282D: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C583E1A: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C583E1A: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C584C37: PyEval_EvalCodeEx (in /usr/lib/libpython2.7.so.1.0)
==30919== 
==30919== Invalid read of size 4
==30919==    at 0x1C6BCF32: PyObject_Free (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C695FD7: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C69607D: PyNode_Free (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C5F1ED7: PyParser_ASTFromString (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C5DAE5C: PyRun_StringFlags (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C580537: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C584C37: PyEval_EvalCodeEx (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C58282D: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C584C37: PyEval_EvalCodeEx (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C58282D: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C583E1A: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C583E1A: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==30919==  Address 0x1cf1d020 is 33,344 bytes inside a block of size 36,032 free'd
==30919==    at 0x4C2A739: free (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==30919==    by 0x1C680527: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C5F1F27: PyParser_ASTFromString (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C5DAE5C: PyRun_StringFlags (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C580537: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C584C37: PyEval_EvalCodeEx (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C58282D: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C584C37: PyEval_EvalCodeEx (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C58282D: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C583E1A: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C583E1A: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C584C37: PyEval_EvalCodeEx (in /usr/lib/libpython2.7.so.1.0)
==30919== 
==30919== Invalid read of size 4
==30919==    at 0x1C695402: PyObject_Realloc (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C6C49D1: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C5DEB40: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C5CA9A7: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C598328: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C59A114: PyAST_Compile (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C5DAE76: PyRun_StringFlags (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C580537: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C584C37: PyEval_EvalCodeEx (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C58282D: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C584C37: PyEval_EvalCodeEx (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C58282D: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==30919==  Address 0x1cf1e020 is 0 bytes after a block of size 32 free'd
==30919==    at 0x4C2B4F0: realloc (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==30919==    by 0x1C607ADD: PyList_Append (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C607BE0: PyArena_AddPyObject (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C5944F8: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C59552C: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C5F3254: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C595B53: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C5F17A6: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C5F1B10: PyAST_FromNode (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C5F1ECA: PyParser_ASTFromString (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C5DAE5C: PyRun_StringFlags (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C580537: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==30919== 
==30919== Invalid read of size 4
==30919==    at 0x1C695402: PyObject_Realloc (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C6C49D1: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C5DEB40: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C5CBA5B: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C5983AB: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C59A114: PyAST_Compile (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C5DAE76: PyRun_StringFlags (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C580537: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C584C37: PyEval_EvalCodeEx (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C58282D: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C584C37: PyEval_EvalCodeEx (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C58282D: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==30919==  Address 0x1cf1e020 is 0 bytes after a block of size 32 free'd
==30919==    at 0x4C2B4F0: realloc (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==30919==    by 0x1C607ADD: PyList_Append (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C607BE0: PyArena_AddPyObject (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C5944F8: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C59552C: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C5F3254: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C595B53: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C5F17A6: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C5F1B10: PyAST_FromNode (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C5F1ECA: PyParser_ASTFromString (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C5DAE5C: PyRun_StringFlags (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C580537: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==30919== 
==30919== Invalid read of size 4
==30919==    at 0x1C6BCF32: PyObject_Free (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C695F14: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C69607D: PyNode_Free (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C5F1ED7: PyParser_ASTFromString (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C5DAE5C: PyRun_StringFlags (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C580537: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C584C37: PyEval_EvalCodeEx (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C58282D: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C584C37: PyEval_EvalCodeEx (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C58282D: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C583E1A: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C583E1A: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==30919==  Address 0x1cf45020 is 32,176 bytes inside a block of size 36,032 free'd
==30919==    at 0x4C2A739: free (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==30919==    by 0x1C680527: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C5F1F27: PyParser_ASTFromString (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C5DAE5C: PyRun_StringFlags (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C580537: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C584C37: PyEval_EvalCodeEx (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C58282D: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C584C37: PyEval_EvalCodeEx (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C58282D: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C583E1A: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C583E1A: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C584C37: PyEval_EvalCodeEx (in /usr/lib/libpython2.7.so.1.0)
==30919== 
==30919== Invalid read of size 4
==30919==    at 0x1C695402: PyObject_Realloc (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C6C49D1: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C5DE8A6: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C5CB707: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C5CA973: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C5CBA5B: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C5CBB29: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C5CA9A7: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C598328: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C59A114: PyAST_Compile (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C5DAE76: PyRun_StringFlags (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C580537: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==30919==  Address 0x1cf47020 is 128 bytes inside a block of size 768 free'd
==30919==    at 0x4C2A739: free (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==30919==    by 0x1C6A369A: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C5E239A: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C5972BF: PySymtable_Build (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C599EB1: PyAST_Compile (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C5DAE76: PyRun_StringFlags (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C580537: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C584C37: PyEval_EvalCodeEx (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C58282D: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C584C37: PyEval_EvalCodeEx (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C58282D: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C583E1A: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==30919== 
==30919== Invalid read of size 4
==30919==    at 0x1C6BCF32: PyObject_Free (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C6A367E: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C6A3466: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C584771: PyEval_EvalCodeEx (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C58282D: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C584C37: PyEval_EvalCodeEx (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C58282D: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C583E1A: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C583E1A: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C584C37: PyEval_EvalCodeEx (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C584D71: PyEval_EvalCode (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C5DBB7C: PyImport_ExecCodeModuleEx (in /usr/lib/libpython2.7.so.1.0)
==30919==  Address 0x1cf3a020 is 6,800 bytes inside a block of size 8,192 free'd
==30919==    at 0x4C2A739: free (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==30919==    by 0x1C698999: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C698A92: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C6ACCCC: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C57EDE5: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C584C37: PyEval_EvalCodeEx (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C58282D: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C584C37: PyEval_EvalCodeEx (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C58282D: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C583E1A: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C583E1A: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C584C37: PyEval_EvalCodeEx (in /usr/lib/libpython2.7.so.1.0)
==30919== 
==30919== Invalid read of size 4
==30919==    at 0x1C6BCF32: PyObject_Free (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C695EEA: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C695EB6: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C695EB6: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C695EB6: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C69607D: PyNode_Free (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C5F1ED7: PyParser_ASTFromString (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C5DAE5C: PyRun_StringFlags (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C580537: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C584C37: PyEval_EvalCodeEx (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C58282D: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C584C37: PyEval_EvalCodeEx (in /usr/lib/libpython2.7.so.1.0)
==30919==  Address 0x1cf5d020 is 35,888 bytes inside a block of size 36,032 free'd
==30919==    at 0x4C2A739: free (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==30919==    by 0x1C680527: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C5F1F27: PyParser_ASTFromString (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C5DAE5C: PyRun_StringFlags (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C580537: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C584C37: PyEval_EvalCodeEx (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C58282D: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C584C37: PyEval_EvalCodeEx (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C58282D: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C583E1A: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C583E1A: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C584C37: PyEval_EvalCodeEx (in /usr/lib/libpython2.7.so.1.0)
==30919== 
==30919== Invalid read of size 4
==30919==    at 0x1C6BCF32: PyObject_Free (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C695EEA: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C69607D: PyNode_Free (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C5F1ED7: PyParser_ASTFromString (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C5DAE5C: PyRun_StringFlags (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C32932C: cmpy_add_node (nodetype.c:88)
==30919==    by 0x1C3263CF: initclawsmail (clawsmailmodule.c:926)
==30919==    by 0x1C32B104: plugin_init (python_plugin.c:669)
==30919==    by 0x6623FB: plugin_load (plugin.c:514)
==30919==    by 0x662A45: plugin_load_all (plugin.c:618)
==30919==    by 0x4D907D: main (main.c:1418)
==30919==  Address 0x1d047020 is 32,432 bytes inside a block of size 36,032 free'd
==30919==    at 0x4C2A739: free (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==30919==    by 0x1C680527: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C5F1F27: PyParser_ASTFromString (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C5DAE5C: PyRun_StringFlags (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C32932C: cmpy_add_node (nodetype.c:88)
==30919==    by 0x1C3263CF: initclawsmail (clawsmailmodule.c:926)
==30919==    by 0x1C32B104: plugin_init (python_plugin.c:669)
==30919==    by 0x6623FB: plugin_load (plugin.c:514)
==30919==    by 0x662A45: plugin_load_all (plugin.c:618)
==30919==    by 0x4D907D: main (main.c:1418)
==30919== 
==30919== Conditional jump or move depends on uninitialised value(s)
==30919==    at 0x1C6BCF3B: PyObject_Free (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C695F79: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C69607D: PyNode_Free (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C5F1ED7: PyParser_ASTFromString (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C5DAE5C: PyRun_StringFlags (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C32932C: cmpy_add_node (nodetype.c:88)
==30919==    by 0x1C3263CF: initclawsmail (clawsmailmodule.c:926)
==30919==    by 0x1C32B104: plugin_init (python_plugin.c:669)
==30919==    by 0x6623FB: plugin_load (plugin.c:514)
==30919==    by 0x662A45: plugin_load_all (plugin.c:618)
==30919==    by 0x4D907D: main (main.c:1418)
==30919== 
==30919== Use of uninitialised value of size 8
==30919==    at 0x1C6BCF53: PyObject_Free (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C695F79: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C69607D: PyNode_Free (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C5F1ED7: PyParser_ASTFromString (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C5DAE5C: PyRun_StringFlags (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C32932C: cmpy_add_node (nodetype.c:88)
==30919==    by 0x1C3263CF: initclawsmail (clawsmailmodule.c:926)
==30919==    by 0x1C32B104: plugin_init (python_plugin.c:669)
==30919==    by 0x6623FB: plugin_load (plugin.c:514)
==30919==    by 0x662A45: plugin_load_all (plugin.c:618)
==30919==    by 0x4D907D: main (main.c:1418)
==30919== 
==30919== Invalid read of size 4
==30919==    at 0x1C6BCF32: PyObject_Free (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C695FA8: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C69607D: PyNode_Free (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C5F1ED7: PyParser_ASTFromString (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C5DAE5C: PyRun_StringFlags (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C32932C: cmpy_add_node (nodetype.c:88)
==30919==    by 0x1C3263CF: initclawsmail (clawsmailmodule.c:926)
==30919==    by 0x1C32B104: plugin_init (python_plugin.c:669)
==30919==    by 0x6623FB: plugin_load (plugin.c:514)
==30919==    by 0x662A45: plugin_load_all (plugin.c:618)
==30919==    by 0x4D907D: main (main.c:1418)
==30919==  Address 0x1d047020 is 32,432 bytes inside a block of size 36,032 free'd
==30919==    at 0x4C2A739: free (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==30919==    by 0x1C680527: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C5F1F27: PyParser_ASTFromString (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C5DAE5C: PyRun_StringFlags (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C32932C: cmpy_add_node (nodetype.c:88)
==30919==    by 0x1C3263CF: initclawsmail (clawsmailmodule.c:926)
==30919==    by 0x1C32B104: plugin_init (python_plugin.c:669)
==30919==    by 0x6623FB: plugin_load (plugin.c:514)
==30919==    by 0x662A45: plugin_load_all (plugin.c:618)
==30919==    by 0x4D907D: main (main.c:1418)
==30919== 
==30919== Invalid read of size 4
==30919==    at 0x1C6BCF32: PyObject_Free (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C6BF74D: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C5995B3: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C599BAF: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C59A114: PyAST_Compile (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C5DAE76: PyRun_StringFlags (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C32932C: cmpy_add_node (nodetype.c:88)
==30919==    by 0x1C3263CF: initclawsmail (clawsmailmodule.c:926)
==30919==    by 0x1C32B104: plugin_init (python_plugin.c:669)
==30919==    by 0x6623FB: plugin_load (plugin.c:514)
==30919==    by 0x662A45: plugin_load_all (plugin.c:618)
==30919==    by 0x4D907D: main (main.c:1418)
==30919==  Address 0x1d093020 is 8 bytes after a block of size 8 free'd
==30919==    at 0x4C2A739: free (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==30919==    by 0x1C6A3544: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C5C9D30: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C5C9F81: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C598F02: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C599BAF: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C59A114: PyAST_Compile (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C5DAE76: PyRun_StringFlags (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C32932C: cmpy_add_node (nodetype.c:88)
==30919==    by 0x1C3263CF: initclawsmail (clawsmailmodule.c:926)
==30919==    by 0x1C32B104: plugin_init (python_plugin.c:669)
==30919==    by 0x6623FB: plugin_load (plugin.c:514)
==30919== 
==30919== Invalid read of size 4
==30919==    at 0x1C695402: PyObject_Realloc (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C6C49D1: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C5CB5AA: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C5CA9A7: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C5975F1: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C59A60F: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C598F97: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C599BAF: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C59A114: PyAST_Compile (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C5DAE76: PyRun_StringFlags (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C32932C: cmpy_add_node (nodetype.c:88)
==30919==    by 0x1C3263CF: initclawsmail (clawsmailmodule.c:926)
==30919==  Address 0x1d093020 is 8 bytes after a block of size 8 free'd
==30919==    at 0x4C2A739: free (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==30919==    by 0x1C6A3544: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C5C9D30: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C5C9F81: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C598F02: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C599BAF: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C59A114: PyAST_Compile (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C5DAE76: PyRun_StringFlags (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C32932C: cmpy_add_node (nodetype.c:88)
==30919==    by 0x1C3263CF: initclawsmail (clawsmailmodule.c:926)
==30919==    by 0x1C32B104: plugin_init (python_plugin.c:669)
==30919==    by 0x6623FB: plugin_load (plugin.c:514)
==30919== 
==30919== Invalid read of size 4
==30919==    at 0x1C6BCF32: PyObject_Free (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C5995B3: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C599BAF: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C59A114: PyAST_Compile (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C5DAE76: PyRun_StringFlags (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C32932C: cmpy_add_node (nodetype.c:88)
==30919==    by 0x1C3263CF: initclawsmail (clawsmailmodule.c:926)
==30919==    by 0x1C32B104: plugin_init (python_plugin.c:669)
==30919==    by 0x6623FB: plugin_load (plugin.c:514)
==30919==    by 0x662A45: plugin_load_all (plugin.c:618)
==30919==    by 0x4D907D: main (main.c:1418)
==30919==  Address 0x1d093020 is 8 bytes after a block of size 8 free'd
==30919==    at 0x4C2A739: free (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==30919==    by 0x1C6A3544: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C5C9D30: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C5C9F81: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C598F02: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C599BAF: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C59A114: PyAST_Compile (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C5DAE76: PyRun_StringFlags (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C32932C: cmpy_add_node (nodetype.c:88)
==30919==    by 0x1C3263CF: initclawsmail (clawsmailmodule.c:926)
==30919==    by 0x1C32B104: plugin_init (python_plugin.c:669)
==30919==    by 0x6623FB: plugin_load (plugin.c:514)
==30919== 
==30919== Invalid read of size 4
==30919==    at 0x1C6BCF32: PyObject_Free (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C5CB99B: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C5CA9A7: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C5975F1: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C598F97: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C599BAF: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C59A114: PyAST_Compile (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C5DAE76: PyRun_StringFlags (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C32932C: cmpy_add_node (nodetype.c:88)
==30919==    by 0x1C3263CF: initclawsmail (clawsmailmodule.c:926)
==30919==    by 0x1C32B104: plugin_init (python_plugin.c:669)
==30919==    by 0x6623FB: plugin_load (plugin.c:514)
==30919==  Address 0x1d096020 is 224 bytes inside a block of size 384 free'd
==30919==    at 0x4C2A739: free (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==30919==    by 0x1C6BF74D: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C5995B3: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C599BAF: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C59A114: PyAST_Compile (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C5DAE76: PyRun_StringFlags (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C32932C: cmpy_add_node (nodetype.c:88)
==30919==    by 0x1C3263CF: initclawsmail (clawsmailmodule.c:926)
==30919==    by 0x1C32B104: plugin_init (python_plugin.c:669)
==30919==    by 0x6623FB: plugin_load (plugin.c:514)
==30919==    by 0x662A45: plugin_load_all (plugin.c:618)
==30919==    by 0x4D907D: main (main.c:1418)
==30919== 
==30919== Invalid read of size 4
==30919==    at 0x1C6BCF32: PyObject_Free (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C6BF74D: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C652DC5: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C599C4C: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C59A114: PyAST_Compile (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C5DAE76: PyRun_StringFlags (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C32932C: cmpy_add_node (nodetype.c:88)
==30919==    by 0x1C3263CF: initclawsmail (clawsmailmodule.c:926)
==30919==    by 0x1C32B104: plugin_init (python_plugin.c:669)
==30919==    by 0x6623FB: plugin_load (plugin.c:514)
==30919==    by 0x662A45: plugin_load_all (plugin.c:618)
==30919==    by 0x4D907D: main (main.c:1418)
==30919==  Address 0x1d096020 is 224 bytes inside a block of size 384 free'd
==30919==    at 0x4C2A739: free (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==30919==    by 0x1C6BF74D: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C5995B3: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C599BAF: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C59A114: PyAST_Compile (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C5DAE76: PyRun_StringFlags (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C32932C: cmpy_add_node (nodetype.c:88)
==30919==    by 0x1C3263CF: initclawsmail (clawsmailmodule.c:926)
==30919==    by 0x1C32B104: plugin_init (python_plugin.c:669)
==30919==    by 0x6623FB: plugin_load (plugin.c:514)
==30919==    by 0x662A45: plugin_load_all (plugin.c:618)
==30919==    by 0x4D907D: main (main.c:1418)
==30919== 
==30919== Invalid read of size 4
==30919==    at 0x1C6BCF32: PyObject_Free (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C65E5B0: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C5DAED1: PyRun_StringFlags (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C32932C: cmpy_add_node (nodetype.c:88)
==30919==    by 0x1C3263CF: initclawsmail (clawsmailmodule.c:926)
==30919==    by 0x1C32B104: plugin_init (python_plugin.c:669)
==30919==    by 0x6623FB: plugin_load (plugin.c:514)
==30919==    by 0x662A45: plugin_load_all (plugin.c:618)
==30919==    by 0x4D907D: main (main.c:1418)
==30919==  Address 0x1d098020 is not stack'd, malloc'd or (recently) free'd
==30919== 
==30919== Invalid read of size 4
==30919==    at 0x1C6BCF32: PyObject_Free (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C696035: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C69607D: PyNode_Free (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C5F1ED7: PyParser_ASTFromString (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C5DAE5C: PyRun_StringFlags (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C3262F9: add_miscstuff (clawsmailmodule.c:898)
==30919==    by 0x1C3264E7: initclawsmail (clawsmailmodule.c:936)
==30919==    by 0x1C32B104: plugin_init (python_plugin.c:669)
==30919==    by 0x6623FB: plugin_load (plugin.c:514)
==30919==    by 0x662A45: plugin_load_all (plugin.c:618)
==30919==    by 0x4D907D: main (main.c:1418)
==30919==  Address 0x1d0a6020 is 32,608 bytes inside a block of size 36,032 free'd
==30919==    at 0x4C2A739: free (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==30919==    by 0x1C680527: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C5F1F27: PyParser_ASTFromString (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C5DAE5C: PyRun_StringFlags (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C3262F9: add_miscstuff (clawsmailmodule.c:898)
==30919==    by 0x1C3264E7: initclawsmail (clawsmailmodule.c:936)
==30919==    by 0x1C32B104: plugin_init (python_plugin.c:669)
==30919==    by 0x6623FB: plugin_load (plugin.c:514)
==30919==    by 0x662A45: plugin_load_all (plugin.c:618)
==30919==    by 0x4D907D: main (main.c:1418)
==30919== 
==30919== Invalid read of size 4
==30919==    at 0x1C6BCF32: PyObject_Free (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C65E5B0: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C5DAED1: PyRun_StringFlags (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C3262F9: add_miscstuff (clawsmailmodule.c:898)
==30919==    by 0x1C3264E7: initclawsmail (clawsmailmodule.c:936)
==30919==    by 0x1C32B104: plugin_init (python_plugin.c:669)
==30919==    by 0x6623FB: plugin_load (plugin.c:514)
==30919==    by 0x662A45: plugin_load_all (plugin.c:618)
==30919==    by 0x4D907D: main (main.c:1418)
==30919==  Address 0x1d0a8020 is 96 bytes inside a block of size 768 free'd
==30919==    at 0x4C2A739: free (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==30919==    by 0x1C6A369A: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C6BF896: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C599FE7: PyAST_Compile (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C5DAE76: PyRun_StringFlags (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C3262F9: add_miscstuff (clawsmailmodule.c:898)
==30919==    by 0x1C3264E7: initclawsmail (clawsmailmodule.c:936)
==30919==    by 0x1C32B104: plugin_init (python_plugin.c:669)
==30919==    by 0x6623FB: plugin_load (plugin.c:514)
==30919==    by 0x662A45: plugin_load_all (plugin.c:618)
==30919==    by 0x4D907D: main (main.c:1418)
==30919== 
==30919== Invalid read of size 4
==30919==    at 0x1C6BCF32: PyObject_Free (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C599FE7: PyAST_Compile (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C5DAE76: PyRun_StringFlags (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C5DBA6A: PyRun_SimpleStringFlags (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C32B137: plugin_init (python_plugin.c:675)
==30919==    by 0x6623FB: plugin_load (plugin.c:514)
==30919==    by 0x662A45: plugin_load_all (plugin.c:618)
==30919==    by 0x4D907D: main (main.c:1418)
==30919==  Address 0x1d0b3020 is 32,304 bytes inside a block of size 36,032 free'd
==30919==    at 0x4C2A739: free (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==30919==    by 0x1C680527: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C5F1F27: PyParser_ASTFromString (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C5DAE5C: PyRun_StringFlags (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C5DBA6A: PyRun_SimpleStringFlags (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C32B137: plugin_init (python_plugin.c:675)
==30919==    by 0x6623FB: plugin_load (plugin.c:514)
==30919==    by 0x662A45: plugin_load_all (plugin.c:618)
==30919==    by 0x4D907D: main (main.c:1418)
==30919== 
==30919== Invalid read of size 4
==30919==    at 0x1C6BCF32: PyObject_Free (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C65E5B0: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C5DAED1: PyRun_StringFlags (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C5DBA6A: PyRun_SimpleStringFlags (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C32B137: plugin_init (python_plugin.c:675)
==30919==    by 0x6623FB: plugin_load (plugin.c:514)
==30919==    by 0x662A45: plugin_load_all (plugin.c:618)
==30919==    by 0x4D907D: main (main.c:1418)
==30919==  Address 0x1d0b4020 is 64 bytes inside a block of size 432 free'd
==30919==    at 0x4C2A739: free (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==30919==    by 0x1C599FE7: PyAST_Compile (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C5DAE76: PyRun_StringFlags (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C5DBA6A: PyRun_SimpleStringFlags (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C32B137: plugin_init (python_plugin.c:675)
==30919==    by 0x6623FB: plugin_load (plugin.c:514)
==30919==    by 0x662A45: plugin_load_all (plugin.c:618)
==30919==    by 0x4D907D: main (main.c:1418)
==30919== 
==30919== Conditional jump or move depends on uninitialised value(s)
==30919==    at 0x1C6BCF3B: PyObject_Free (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C695EEA: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C69607D: PyNode_Free (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C5F1ED7: PyParser_ASTFromString (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C5DAE5C: PyRun_StringFlags (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C5DBA6A: PyRun_SimpleStringFlags (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C32BA85: parasite_python_init (python-hooks.c:156)
==30919==    by 0x1C32B160: plugin_init (python_plugin.c:681)
==30919==    by 0x6623FB: plugin_load (plugin.c:514)
==30919==    by 0x662A45: plugin_load_all (plugin.c:618)
==30919==    by 0x4D907D: main (main.c:1418)
==30919== 
==30919== Use of uninitialised value of size 8
==30919==    at 0x1C6BCF53: PyObject_Free (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C695EEA: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C69607D: PyNode_Free (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C5F1ED7: PyParser_ASTFromString (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C5DAE5C: PyRun_StringFlags (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C5DBA6A: PyRun_SimpleStringFlags (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C32BA85: parasite_python_init (python-hooks.c:156)
==30919==    by 0x1C32B160: plugin_init (python_plugin.c:681)
==30919==    by 0x6623FB: plugin_load (plugin.c:514)
==30919==    by 0x662A45: plugin_load_all (plugin.c:618)
==30919==    by 0x4D907D: main (main.c:1418)
==30919== 
==30919== Conditional jump or move depends on uninitialised value(s)
==30919==    at 0x1C6BCF3B: PyObject_Free (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C696035: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C695EB6: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C69607D: PyNode_Free (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C5F1ED7: PyParser_ASTFromString (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C5DAE5C: PyRun_StringFlags (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C5DBA6A: PyRun_SimpleStringFlags (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C32BA85: parasite_python_init (python-hooks.c:156)
==30919==    by 0x1C32B160: plugin_init (python_plugin.c:681)
==30919==    by 0x6623FB: plugin_load (plugin.c:514)
==30919==    by 0x662A45: plugin_load_all (plugin.c:618)
==30919==    by 0x4D907D: main (main.c:1418)
==30919== 
==30919== Use of uninitialised value of size 8
==30919==    at 0x1C6BCF53: PyObject_Free (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C696035: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C695EB6: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C69607D: PyNode_Free (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C5F1ED7: PyParser_ASTFromString (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C5DAE5C: PyRun_StringFlags (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C5DBA6A: PyRun_SimpleStringFlags (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C32BA85: parasite_python_init (python-hooks.c:156)
==30919==    by 0x1C32B160: plugin_init (python_plugin.c:681)
==30919==    by 0x6623FB: plugin_load (plugin.c:514)
==30919==    by 0x662A45: plugin_load_all (plugin.c:618)
==30919==    by 0x4D907D: main (main.c:1418)
==30919== 
==30919== Invalid read of size 4
==30919==    at 0x1C6BCF32: PyObject_Free (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C695F79: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C69607D: PyNode_Free (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C5F1ED7: PyParser_ASTFromString (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C5DAE5C: PyRun_StringFlags (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C5DBA6A: PyRun_SimpleStringFlags (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C32BA85: parasite_python_init (python-hooks.c:156)
==30919==    by 0x1C32B160: plugin_init (python_plugin.c:681)
==30919==    by 0x6623FB: plugin_load (plugin.c:514)
==30919==    by 0x662A45: plugin_load_all (plugin.c:618)
==30919==    by 0x4D907D: main (main.c:1418)
==30919==  Address 0x1d0c1020 is 34,080 bytes inside a block of size 36,032 free'd
==30919==    at 0x4C2A739: free (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==30919==    by 0x1C680527: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C5F1F27: PyParser_ASTFromString (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C5DAE5C: PyRun_StringFlags (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C5DBA6A: PyRun_SimpleStringFlags (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C32BA85: parasite_python_init (python-hooks.c:156)
==30919==    by 0x1C32B160: plugin_init (python_plugin.c:681)
==30919==    by 0x6623FB: plugin_load (plugin.c:514)
==30919==    by 0x662A45: plugin_load_all (plugin.c:618)
==30919==    by 0x4D907D: main (main.c:1418)
==30919== 
==30919== Invalid read of size 4
==30919==    at 0x1C6BCF32: PyObject_Free (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C652DC5: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C599C4C: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C59A114: PyAST_Compile (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C5DAE76: PyRun_StringFlags (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C5DBA6A: PyRun_SimpleStringFlags (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C32BA85: parasite_python_init (python-hooks.c:156)
==30919==    by 0x1C32B160: plugin_init (python_plugin.c:681)
==30919==    by 0x6623FB: plugin_load (plugin.c:514)
==30919==    by 0x662A45: plugin_load_all (plugin.c:618)
==30919==    by 0x4D907D: main (main.c:1418)
==30919==  Address 0x1d0c5020 is 64 bytes inside a block of size 68 free'd
==30919==    at 0x4C2A739: free (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==30919==    by 0x1C5E4B94: PyCode_Optimize (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C5E625E: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C5995A3: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C599BAF: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C59A114: PyAST_Compile (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C5DAE76: PyRun_StringFlags (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C5DBA6A: PyRun_SimpleStringFlags (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C32BA85: parasite_python_init (python-hooks.c:156)
==30919==    by 0x1C32B160: plugin_init (python_plugin.c:681)
==30919==    by 0x6623FB: plugin_load (plugin.c:514)
==30919==    by 0x662A45: plugin_load_all (plugin.c:618)
==30919== 
==30919== Conditional jump or move depends on uninitialised value(s)
==30919==    at 0x1C6BCF3B: PyObject_Free (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C65E5B0: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C680B8E: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C5E06C0: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C6812EB: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C638689: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C6817FC: PyImport_ImportModuleLevel (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C5D703E: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C6541FD: PyObject_Call (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C654B06: PyEval_CallObjectWithKeywords (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C57F5C2: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C584C37: PyEval_EvalCodeEx (in /usr/lib/libpython2.7.so.1.0)
==30919== 
==30919== Use of uninitialised value of size 8
==30919==    at 0x1C6BCF53: PyObject_Free (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C65E5B0: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C680B8E: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C5E06C0: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C6812EB: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C638689: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C6817FC: PyImport_ImportModuleLevel (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C5D703E: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C6541FD: PyObject_Call (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C654B06: PyEval_CallObjectWithKeywords (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C57F5C2: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C584C37: PyEval_EvalCodeEx (in /usr/lib/libpython2.7.so.1.0)
==30919== 
==30919== Conditional jump or move depends on uninitialised value(s)
==30919==    at 0x1C69540B: PyObject_Realloc (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C6C63F8: _PyObject_GC_Resize (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C59220B: PyFrame_New (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C583DC9: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C583E1A: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C583E1A: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C583E1A: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C583E1A: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C583E1A: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C584C37: PyEval_EvalCodeEx (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C58282D: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C584C37: PyEval_EvalCodeEx (in /usr/lib/libpython2.7.so.1.0)
==30919== 
==30919== Use of uninitialised value of size 8
==30919==    at 0x1C69541F: PyObject_Realloc (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C6C63F8: _PyObject_GC_Resize (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C59220B: PyFrame_New (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C583DC9: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C583E1A: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C583E1A: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C583E1A: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C583E1A: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C583E1A: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C584C37: PyEval_EvalCodeEx (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C58282D: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C584C37: PyEval_EvalCodeEx (in /usr/lib/libpython2.7.so.1.0)
==30919== 
==30919== Invalid read of size 4
==30919==    at 0x1C695402: PyObject_Realloc (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C6C63F8: _PyObject_GC_Resize (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C59220B: PyFrame_New (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C583DC9: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C583E1A: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C583E1A: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C584C37: PyEval_EvalCodeEx (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C58282D: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C584C37: PyEval_EvalCodeEx (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C58282D: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C584C37: PyEval_EvalCodeEx (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C58282D: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==30919==  Address 0x1ce5b020 is not stack'd, malloc'd or (recently) free'd
==30919== 
==30919== Invalid read of size 4
==30919==    at 0x1C6BCF32: PyObject_Free (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C65E5DA: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C680B8E: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C5E06C0: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C6812EB: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C638612: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C6817FC: PyImport_ImportModuleLevel (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C5D703E: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C6541FD: PyObject_Call (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C655F4A: PyObject_CallFunction (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C65A7D9: PyImport_Import (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C65BF09: PyImport_ImportModule (in /usr/lib/libpython2.7.so.1.0)
==30919==  Address 0x1d0d8020 is 1,456 bytes inside a block of size 5,410 free'd
==30919==    at 0x4C2A739: free (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==30919==    by 0x1C67B36A: PyMarshal_ReadLastObjectFromFile (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C680A0D: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C5E06C0: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C6812EB: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C638612: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C6817FC: PyImport_ImportModuleLevel (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C5D703E: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C6541FD: PyObject_Call (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C655F4A: PyObject_CallFunction (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C65A7D9: PyImport_Import (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C65BF09: PyImport_ImportModule (in /usr/lib/libpython2.7.so.1.0)
==30919== 
==30919== Invalid read of size 4
==30919==    at 0x1C6BCF32: PyObject_Free (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C6A37CA: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C6A37B1: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C65E65A: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C680B8E: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C5E06C0: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C6812EB: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C638612: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C6817FC: PyImport_ImportModuleLevel (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C5D703E: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C6541FD: PyObject_Call (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C655F4A: PyObject_CallFunction (in /usr/lib/libpython2.7.so.1.0)
==30919==  Address 0x1d0d9020 is 64 bytes inside a block of size 613 free'd
==30919==    at 0x4C2A739: free (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==30919==    by 0x1C65E5DA: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C680B8E: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C5E06C0: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C6812EB: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C638612: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C6817FC: PyImport_ImportModuleLevel (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C5D703E: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C6541FD: PyObject_Call (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C655F4A: PyObject_CallFunction (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C65A7D9: PyImport_Import (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C65BF09: PyImport_ImportModule (in /usr/lib/libpython2.7.so.1.0)
==30919== 
==30919== Invalid read of size 4
==30919==    at 0x1C6BCF32: PyObject_Free (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C65E5B0: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C680B8E: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C5E06C0: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C6812EB: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C638612: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C6817FC: PyImport_ImportModuleLevel (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C5D703E: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C6541FD: PyObject_Call (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C655F4A: PyObject_CallFunction (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C65A7D9: PyImport_Import (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C65BF09: PyImport_ImportModule (in /usr/lib/libpython2.7.so.1.0)
==30919==  Address 0x1d0db020 is 96 bytes inside a block of size 1,184 free'd
==30919==    at 0x4C2A739: free (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==30919==    by 0x1C6A3544: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C67B273: PyMarshal_ReadObjectFromString (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C67B35D: PyMarshal_ReadLastObjectFromFile (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C680A0D: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C5E06C0: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C6812EB: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C638612: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C6817FC: PyImport_ImportModuleLevel (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C5D703E: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C6541FD: PyObject_Call (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C655F4A: PyObject_CallFunction (in /usr/lib/libpython2.7.so.1.0)
==30919== 
==30938== 
==30938== HEAP SUMMARY:
==30938==     in use at exit: 13,046,846 bytes in 71,256 blocks
==30938==   total heap usage: 246,913 allocs, 175,657 frees, 35,598,488 bytes allocated
==30938== 
==30938== LEAK SUMMARY:
==30938==    definitely lost: 41,107 bytes in 112 blocks
==30938==    indirectly lost: 10,192 bytes in 320 blocks
==30938==      possibly lost: 3,343,827 bytes in 20,556 blocks
==30938==    still reachable: 9,651,720 bytes in 50,268 blocks
==30938==         suppressed: 0 bytes in 0 blocks
==30938== Rerun with --leak-check=full to see details of leaked memory
==30938== 
==30938== For counts of detected and suppressed errors, rerun with: -v
==30938== Use --track-origins=yes to see where uninitialised values come from
==30938== ERROR SUMMARY: 1815 errors from 75 contexts (suppressed: 2 from 2)
==30940== 
==30940== HEAP SUMMARY:
==30940==     in use at exit: 13,046,916 bytes in 71,259 blocks
==30940==   total heap usage: 246,916 allocs, 175,657 frees, 35,598,558 bytes allocated
==30940== 
==30941== 
==30941== HEAP SUMMARY:
==30941==     in use at exit: 13,046,916 bytes in 71,259 blocks
==30941==   total heap usage: 246,916 allocs, 175,657 frees, 35,598,558 bytes allocated
==30941== 
==30940== LEAK SUMMARY:
==30940==    definitely lost: 41,107 bytes in 112 blocks
==30940==    indirectly lost: 10,192 bytes in 320 blocks
==30940==      possibly lost: 3,343,827 bytes in 20,556 blocks
==30940==    still reachable: 9,651,790 bytes in 50,271 blocks
==30940==         suppressed: 0 bytes in 0 blocks
==30940== Rerun with --leak-check=full to see details of leaked memory
==30940== 
==30940== For counts of detected and suppressed errors, rerun with: -v
==30940== Use --track-origins=yes to see where uninitialised values come from
==30940== ERROR SUMMARY: 1815 errors from 75 contexts (suppressed: 2 from 2)
==30941== LEAK SUMMARY:
==30941==    definitely lost: 41,107 bytes in 112 blocks
==30941==    indirectly lost: 10,192 bytes in 320 blocks
==30941==      possibly lost: 3,343,827 bytes in 20,556 blocks
==30941==    still reachable: 9,651,790 bytes in 50,271 blocks
==30941==         suppressed: 0 bytes in 0 blocks
==30941== Rerun with --leak-check=full to see details of leaked memory
==30941== 
==30941== For counts of detected and suppressed errors, rerun with: -v
==30941== Use --track-origins=yes to see where uninitialised values come from
==30941== ERROR SUMMARY: 1815 errors from 75 contexts (suppressed: 2 from 2)
==30942== 
==30943== 
==30942== HEAP SUMMARY:
==30942==     in use at exit: 13,046,981 bytes in 71,261 blocks
==30942==   total heap usage: 246,918 allocs, 175,657 frees, 35,598,623 bytes allocated
==30942== 
==30943== HEAP SUMMARY:
==30943==     in use at exit: 13,046,981 bytes in 71,261 blocks
==30943==   total heap usage: 246,918 allocs, 175,657 frees, 35,598,623 bytes allocated
==30943== 
==30942== LEAK SUMMARY:
==30942==    definitely lost: 41,107 bytes in 112 blocks
==30942==    indirectly lost: 10,192 bytes in 320 blocks
==30942==      possibly lost: 3,343,827 bytes in 20,556 blocks
==30942==    still reachable: 9,651,855 bytes in 50,273 blocks
==30942==         suppressed: 0 bytes in 0 blocks
==30942== Rerun with --leak-check=full to see details of leaked memory
==30942== 
==30942== For counts of detected and suppressed errors, rerun with: -v
==30942== Use --track-origins=yes to see where uninitialised values come from
==30942== ERROR SUMMARY: 1815 errors from 75 contexts (suppressed: 2 from 2)
==30943== LEAK SUMMARY:
==30943==    definitely lost: 41,107 bytes in 112 blocks
==30943==    indirectly lost: 10,192 bytes in 320 blocks
==30943==      possibly lost: 3,343,827 bytes in 20,556 blocks
==30943==    still reachable: 9,651,855 bytes in 50,273 blocks
==30943==         suppressed: 0 bytes in 0 blocks
==30943== Rerun with --leak-check=full to see details of leaked memory
==30943== 
==30943== For counts of detected and suppressed errors, rerun with: -v
==30943== Use --track-origins=yes to see where uninitialised values come from
==30943== ERROR SUMMARY: 1815 errors from 75 contexts (suppressed: 2 from 2)
==30944== 
==30944== HEAP SUMMARY:
==30944==     in use at exit: 13,046,981 bytes in 71,261 blocks
==30944==   total heap usage: 246,918 allocs, 175,657 frees, 35,598,623 bytes allocated
==30944== 
==30945== 
==30945== HEAP SUMMARY:
==30945==     in use at exit: 13,046,981 bytes in 71,261 blocks
==30945==   total heap usage: 246,918 allocs, 175,657 frees, 35,598,623 bytes allocated
==30945== 
==30944== LEAK SUMMARY:
==30944==    definitely lost: 41,107 bytes in 112 blocks
==30944==    indirectly lost: 10,192 bytes in 320 blocks
==30944==      possibly lost: 3,343,827 bytes in 20,556 blocks
==30944==    still reachable: 9,651,855 bytes in 50,273 blocks
==30944==         suppressed: 0 bytes in 0 blocks
==30944== Rerun with --leak-check=full to see details of leaked memory
==30944== 
==30944== For counts of detected and suppressed errors, rerun with: -v
==30944== Use --track-origins=yes to see where uninitialised values come from
==30944== ERROR SUMMARY: 1815 errors from 75 contexts (suppressed: 2 from 2)
==30945== LEAK SUMMARY:
==30945==    definitely lost: 41,107 bytes in 112 blocks
==30945==    indirectly lost: 10,192 bytes in 320 blocks
==30945==      possibly lost: 3,343,827 bytes in 20,556 blocks
==30945==    still reachable: 9,651,855 bytes in 50,273 blocks
==30945==         suppressed: 0 bytes in 0 blocks
==30945== Rerun with --leak-check=full to see details of leaked memory
==30945== 
==30945== For counts of detected and suppressed errors, rerun with: -v
==30945== Use --track-origins=yes to see where uninitialised values come from
==30945== ERROR SUMMARY: 1815 errors from 75 contexts (suppressed: 2 from 2)
==30919== Conditional jump or move depends on uninitialised value(s)
==30919==    at 0xB43C9F0: ??? (in /usr/lib/x86_64-linux-gnu/libpixman-1.so.0.30.2)
==30919==    by 0xB422076: ??? (in /usr/lib/x86_64-linux-gnu/libpixman-1.so.0.30.2)
==30919==    by 0xB3D5810: pixman_image_composite32 (in /usr/lib/x86_64-linux-gnu/libpixman-1.so.0.30.2)
==30919==    by 0x6513C6D: ??? (in /usr/lib/x86_64-linux-gnu/libcairo.so.2.11200.2)
==30919==    by 0x6550CA6: ??? (in /usr/lib/x86_64-linux-gnu/libcairo.so.2.11200.2)
==30919==    by 0x654362A: ??? (in /usr/lib/x86_64-linux-gnu/libcairo.so.2.11200.2)
==30919==    by 0x6543F6E: ??? (in /usr/lib/x86_64-linux-gnu/libcairo.so.2.11200.2)
==30919==    by 0x6544A6E: ??? (in /usr/lib/x86_64-linux-gnu/libcairo.so.2.11200.2)
==30919==    by 0x65086ED: ??? (in /usr/lib/x86_64-linux-gnu/libcairo.so.2.11200.2)
==30919==    by 0x6518932: ??? (in /usr/lib/x86_64-linux-gnu/libcairo.so.2.11200.2)
==30919==    by 0x6547C4D: ??? (in /usr/lib/x86_64-linux-gnu/libcairo.so.2.11200.2)
==30919==    by 0x650FE24: ??? (in /usr/lib/x86_64-linux-gnu/libcairo.so.2.11200.2)
==30919== 
==30919== Conditional jump or move depends on uninitialised value(s)
==30919==    at 0xB43C6E8: ??? (in /usr/lib/x86_64-linux-gnu/libpixman-1.so.0.30.2)
==30919==    by 0xB422076: ??? (in /usr/lib/x86_64-linux-gnu/libpixman-1.so.0.30.2)
==30919==    by 0xB3D5810: pixman_image_composite32 (in /usr/lib/x86_64-linux-gnu/libpixman-1.so.0.30.2)
==30919==    by 0x6513C6D: ??? (in /usr/lib/x86_64-linux-gnu/libcairo.so.2.11200.2)
==30919==    by 0x6550CA6: ??? (in /usr/lib/x86_64-linux-gnu/libcairo.so.2.11200.2)
==30919==    by 0x654362A: ??? (in /usr/lib/x86_64-linux-gnu/libcairo.so.2.11200.2)
==30919==    by 0x6543F6E: ??? (in /usr/lib/x86_64-linux-gnu/libcairo.so.2.11200.2)
==30919==    by 0x6544A6E: ??? (in /usr/lib/x86_64-linux-gnu/libcairo.so.2.11200.2)
==30919==    by 0x65086ED: ??? (in /usr/lib/x86_64-linux-gnu/libcairo.so.2.11200.2)
==30919==    by 0x6518932: ??? (in /usr/lib/x86_64-linux-gnu/libcairo.so.2.11200.2)
==30919==    by 0x6547C4D: ??? (in /usr/lib/x86_64-linux-gnu/libcairo.so.2.11200.2)
==30919==    by 0x650FE24: ??? (in /usr/lib/x86_64-linux-gnu/libcairo.so.2.11200.2)
==30919== 
==30919== Conditional jump or move depends on uninitialised value(s)
==30919==    at 0x334590A7: ??? (in /usr/lib/x86_64-linux-gnu/librsvg-2.so.2.36.3)
==30919==    by 0x33459D48: rsvg_handle_get_pixbuf_sub (in /usr/lib/x86_64-linux-gnu/librsvg-2.so.2.36.3)
==30919==    by 0x33201EAA: ??? (in /usr/lib/x86_64-linux-gnu/gdk-pixbuf-2.0/2.10.0/loaders/libpixbufloader-svg.so)
==30919==    by 0x62D272B: gdk_pixbuf_loader_close (in /usr/lib/x86_64-linux-gnu/libgdk_pixbuf-2.0.so.0.2600.4)
==30919==    by 0x62CEA7A: ??? (in /usr/lib/x86_64-linux-gnu/libgdk_pixbuf-2.0.so.0.2600.4)
==30919==    by 0x62D02B0: gdk_pixbuf_new_from_stream_at_scale (in /usr/lib/x86_64-linux-gnu/libgdk_pixbuf-2.0.so.0.2600.4)
==30919==    by 0x557F5AD: ??? (in /usr/lib/x86_64-linux-gnu/libgtk-x11-2.0.so.0.2400.13)
==30919==    by 0x5581FB9: gtk_icon_info_load_icon (in /usr/lib/x86_64-linux-gnu/libgtk-x11-2.0.so.0.2400.13)
==30919==    by 0x5582166: gtk_icon_theme_load_icon (in /usr/lib/x86_64-linux-gnu/libgtk-x11-2.0.so.0.2400.13)
==30919==    by 0x557D650: gtk_icon_set_render_icon (in /usr/lib/x86_64-linux-gnu/libgtk-x11-2.0.so.0.2400.13)
==30919==    by 0x56D04B2: gtk_widget_render_icon (in /usr/lib/x86_64-linux-gnu/libgtk-x11-2.0.so.0.2400.13)
==30919==    by 0x559165B: ??? (in /usr/lib/x86_64-linux-gnu/libgtk-x11-2.0.so.0.2400.13)
==30919== 
==30919== Conditional jump or move depends on uninitialised value(s)
==30919==    at 0x89DBDB0: re_compile_fastmap_iter.isra.30 (regcomp.c:329)
==30919==    by 0x89E8669: re_compile_fastmap (regcomp.c:281)
==30919==    by 0x89E8E8F: regcomp (regcomp.c:517)
==30919==    by 0x67C2F8: subject_get_prefix_length (utils.c:3805)
==30919==    by 0x59CB84: subject_hashtable_insert (procmsg.c:141)
==30919==    by 0x59CF2D: procmsg_get_thread_tree (procmsg.c:242)
==30919==    by 0x5BFFE2: summary_set_ctree_from_list (summaryview.c:3141)
==30919==    by 0x5BA939: summary_show (summaryview.c:1463)
==30919==    by 0x4B6B7E: folderview_selected (folderview.c:2145)
==30919==    by 0x6FD98A: claws_marshal_VOID__POINTER_INT (claws-marshal.c:348)
==30919==    by 0x7E0C13F: g_closure_invoke (in /usr/lib/x86_64-linux-gnu/libgobject-2.0.so.0.3400.1)
==30919==    by 0x7E1D54F: ??? (in /usr/lib/x86_64-linux-gnu/libgobject-2.0.so.0.3400.1)
==30919== 
==30919== Conditional jump or move depends on uninitialised value(s)
==30919==    at 0x89DBDB0: re_compile_fastmap_iter.isra.30 (regcomp.c:329)
==30919==    by 0x89E8689: re_compile_fastmap (regcomp.c:283)
==30919==    by 0x89E8E8F: regcomp (regcomp.c:517)
==30919==    by 0x67C2F8: subject_get_prefix_length (utils.c:3805)
==30919==    by 0x59CB84: subject_hashtable_insert (procmsg.c:141)
==30919==    by 0x59CF2D: procmsg_get_thread_tree (procmsg.c:242)
==30919==    by 0x5BFFE2: summary_set_ctree_from_list (summaryview.c:3141)
==30919==    by 0x5BA939: summary_show (summaryview.c:1463)
==30919==    by 0x4B6B7E: folderview_selected (folderview.c:2145)
==30919==    by 0x6FD98A: claws_marshal_VOID__POINTER_INT (claws-marshal.c:348)
==30919==    by 0x7E0C13F: g_closure_invoke (in /usr/lib/x86_64-linux-gnu/libgobject-2.0.so.0.3400.1)
==30919==    by 0x7E1D54F: ??? (in /usr/lib/x86_64-linux-gnu/libgobject-2.0.so.0.3400.1)
==30919== 
==30919== Invalid read of size 4
==30919==    at 0x1FA5EA49: SwapDWord (ytnef.c:165)
==30919==    by 0x1FA6009A: TNEFPriority (ytnef.c:660)
==30919==    by 0x1FA61892: TNEFParse (ytnef.c:1060)
==30919==    by 0x1FA61176: TNEFParseFile (ytnef.c:923)
==30919==    by 0x1FA5B6E0: tnef_parse (tnef_parse.c:278)
==30919==    by 0x59AACE: procmime_mimeparser_parse (procmime.c:2000)
==30919==    by 0x59AD47: procmime_parse_mimepart (procmime.c:2086)
==30919==    by 0x59940A: procmime_parse_multipart (procmime.c:1677)
==30919==    by 0x59AE27: procmime_parse_mimepart (procmime.c:2107)
==30919==    by 0x598C4F: procmime_parse_message_rfc822 (procmime.c:1501)
==30919==    by 0x59B089: procmime_scan_file_with_offset (procmime.c:2174)
==30919==    by 0x59B1D8: procmime_scan_file_full (procmime.c:2187)
==30919==  Address 0x366d4960 is 0 bytes inside a block of size 2 alloc'd
==30919==    at 0x4C29E46: calloc (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==30919==    by 0x1FA616B2: TNEFParse (ytnef.c:1031)
==30919==    by 0x1FA61176: TNEFParseFile (ytnef.c:923)
==30919==    by 0x1FA5B6E0: tnef_parse (tnef_parse.c:278)
==30919==    by 0x59AACE: procmime_mimeparser_parse (procmime.c:2000)
==30919==    by 0x59AD47: procmime_parse_mimepart (procmime.c:2086)
==30919==    by 0x59940A: procmime_parse_multipart (procmime.c:1677)
==30919==    by 0x59AE27: procmime_parse_mimepart (procmime.c:2107)
==30919==    by 0x598C4F: procmime_parse_message_rfc822 (procmime.c:1501)
==30919==    by 0x59B089: procmime_scan_file_with_offset (procmime.c:2174)
==30919==    by 0x59B1D8: procmime_scan_file_full (procmime.c:2187)
==30919==    by 0x59B205: procmime_scan_file (procmime.c:2194)
==30919== 
==30919== Invalid read of size 8
==30919==    at 0x89AAA0F: wcslen (wcslen.S:48)
==30919==    by 0x89B4D99: wcscoll_l (strcoll_l.c:509)
==30919==    by 0x80BDE79: g_utf8_collate (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.3400.1)
==30919==    by 0x597DE3: procmime_str_equal (procmime.c:1143)
==30919==    by 0x8082A7E: g_hash_table_lookup (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.3400.1)
==30919==    by 0x597EB7: procmime_get_mime_type_table (procmime.c:1171)
==30919==    by 0x597C0B: procmime_get_mime_type (procmime.c:1092)
==30919==    by 0x1FA5ADA9: tnef_dump_file (tnef_parse.c:101)
==30919==    by 0x1FA5B33A: tnef_parse_rtf (tnef_parse.c:205)
==30919==    by 0x1FA5B894: tnef_parse (tnef_parse.c:311)
==30919==    by 0x59AACE: procmime_mimeparser_parse (procmime.c:2000)
==30919==    by 0x59AD47: procmime_parse_mimepart (procmime.c:2086)
==30919==  Address 0x3690a1e0 is 32 bytes inside a block of size 36 alloc'd
==30919==    at 0x4C2B3F8: malloc (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==30919==    by 0x80986E0: g_malloc (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.3400.1)
==30919==    by 0x80BEB20: ??? (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.3400.1)
==30919==    by 0x80BDE54: g_utf8_collate (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.3400.1)
==30919==    by 0x597DE3: procmime_str_equal (procmime.c:1143)
==30919==    by 0x8082A7E: g_hash_table_lookup (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.3400.1)
==30919==    by 0x597EB7: procmime_get_mime_type_table (procmime.c:1171)
==30919==    by 0x597C0B: procmime_get_mime_type (procmime.c:1092)
==30919==    by 0x1FA5ADA9: tnef_dump_file (tnef_parse.c:101)
==30919==    by 0x1FA5B33A: tnef_parse_rtf (tnef_parse.c:205)
==30919==    by 0x1FA5B894: tnef_parse (tnef_parse.c:311)
==30919==    by 0x59AACE: procmime_mimeparser_parse (procmime.c:2000)
==30919== 
==30919== Invalid read of size 8
==30919==    at 0x89AAA0F: wcslen (wcslen.S:48)
==30919==    by 0x89B4DA8: wcscoll_l (strcoll_l.c:510)
==30919==    by 0x80BDE79: g_utf8_collate (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.3400.1)
==30919==    by 0x597DE3: procmime_str_equal (procmime.c:1143)
==30919==    by 0x8082A7E: g_hash_table_lookup (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.3400.1)
==30919==    by 0x597EB7: procmime_get_mime_type_table (procmime.c:1171)
==30919==    by 0x597C0B: procmime_get_mime_type (procmime.c:1092)
==30919==    by 0x1FA5ADA9: tnef_dump_file (tnef_parse.c:101)
==30919==    by 0x1FA5B33A: tnef_parse_rtf (tnef_parse.c:205)
==30919==    by 0x1FA5B894: tnef_parse (tnef_parse.c:311)
==30919==    by 0x59AACE: procmime_mimeparser_parse (procmime.c:2000)
==30919==    by 0x59AD47: procmime_parse_mimepart (procmime.c:2086)
==30919==  Address 0x3690a250 is 32 bytes inside a block of size 36 alloc'd
==30919==    at 0x4C2B3F8: malloc (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==30919==    by 0x80986E0: g_malloc (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.3400.1)
==30919==    by 0x80BEB20: ??? (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.3400.1)
==30919==    by 0x80BDE6B: g_utf8_collate (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.3400.1)
==30919==    by 0x597DE3: procmime_str_equal (procmime.c:1143)
==30919==    by 0x8082A7E: g_hash_table_lookup (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.3400.1)
==30919==    by 0x597EB7: procmime_get_mime_type_table (procmime.c:1171)
==30919==    by 0x597C0B: procmime_get_mime_type (procmime.c:1092)
==30919==    by 0x1FA5ADA9: tnef_dump_file (tnef_parse.c:101)
==30919==    by 0x1FA5B33A: tnef_parse_rtf (tnef_parse.c:205)
==30919==    by 0x1FA5B894: tnef_parse (tnef_parse.c:311)
==30919==    by 0x59AACE: procmime_mimeparser_parse (procmime.c:2000)
==30919== 
==30919== Invalid read of size 8
==30919==    at 0x1FA5CCBF: SaveVCalendar (tnef_dump.c:392)
==30919==    by 0x1FA5B07E: tnef_parse_vcal (tnef_parse.c:147)
==30919==    by 0x1FA5B912: tnef_parse (tnef_parse.c:322)
==30919==    by 0x59AACE: procmime_mimeparser_parse (procmime.c:2000)
==30919==    by 0x59AD47: procmime_parse_mimepart (procmime.c:2086)
==30919==    by 0x59940A: procmime_parse_multipart (procmime.c:1677)
==30919==    by 0x59AE27: procmime_parse_mimepart (procmime.c:2107)
==30919==    by 0x598C4F: procmime_parse_message_rfc822 (procmime.c:1501)
==30919==    by 0x59B089: procmime_scan_file_with_offset (procmime.c:2174)
==30919==    by 0x59B1D8: procmime_scan_file_full (procmime.c:2187)
==30919==    by 0x59B205: procmime_scan_file (procmime.c:2194)
==30919==    by 0x502B45: messageview_show (messageview.c:1337)
==30919==  Address 0x36c2c320 is 0 bytes inside a block of size 4 alloc'd
==30919==    at 0x4C29E46: calloc (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==30919==    by 0x1FA5F658: TNEFFillMapi (ytnef.c:478)
==30919==    by 0x1FA5F166: TNEFMapiProperties (ytnef.c:358)
==30919==    by 0x1FA61892: TNEFParse (ytnef.c:1060)
==30919==    by 0x1FA61176: TNEFParseFile (ytnef.c:923)
==30919==    by 0x1FA5B6E0: tnef_parse (tnef_parse.c:278)
==30919==    by 0x59AACE: procmime_mimeparser_parse (procmime.c:2000)
==30919==    by 0x59AD47: procmime_parse_mimepart (procmime.c:2086)
==30919==    by 0x59940A: procmime_parse_multipart (procmime.c:1677)
==30919==    by 0x59AE27: procmime_parse_mimepart (procmime.c:2107)
==30919==    by 0x598C4F: procmime_parse_message_rfc822 (procmime.c:1501)
==30919==    by 0x59B089: procmime_scan_file_with_offset (procmime.c:2174)
==30919== 
==30919== Invalid read of size 8
==30919==    at 0x1FA5EA61: SwapDDWord (ytnef.c:185)
==30919==    by 0x1FA5D468: SaveVCalendar (tnef_dump.c:556)
==30919==    by 0x1FA5B07E: tnef_parse_vcal (tnef_parse.c:147)
==30919==    by 0x1FA5B912: tnef_parse (tnef_parse.c:322)
==30919==    by 0x59AACE: procmime_mimeparser_parse (procmime.c:2000)
==30919==    by 0x59AD47: procmime_parse_mimepart (procmime.c:2086)
==30919==    by 0x59940A: procmime_parse_multipart (procmime.c:1677)
==30919==    by 0x59AE27: procmime_parse_mimepart (procmime.c:2107)
==30919==    by 0x598C4F: procmime_parse_message_rfc822 (procmime.c:1501)
==30919==    by 0x59B089: procmime_scan_file_with_offset (procmime.c:2174)
==30919==    by 0x59B1D8: procmime_scan_file_full (procmime.c:2187)
==30919==    by 0x59B205: procmime_scan_file (procmime.c:2194)
==30919==  Address 0x36d745f0 is 0 bytes inside a block of size 4 alloc'd
==30919==    at 0x4C29E46: calloc (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==30919==    by 0x1FA5F658: TNEFFillMapi (ytnef.c:478)
==30919==    by 0x1FA5F166: TNEFMapiProperties (ytnef.c:358)
==30919==    by 0x1FA61892: TNEFParse (ytnef.c:1060)
==30919==    by 0x1FA61176: TNEFParseFile (ytnef.c:923)
==30919==    by 0x1FA5B6E0: tnef_parse (tnef_parse.c:278)
==30919==    by 0x59AACE: procmime_mimeparser_parse (procmime.c:2000)
==30919==    by 0x59AD47: procmime_parse_mimepart (procmime.c:2086)
==30919==    by 0x59940A: procmime_parse_multipart (procmime.c:1677)
==30919==    by 0x59AE27: procmime_parse_mimepart (procmime.c:2107)
==30919==    by 0x598C4F: procmime_parse_message_rfc822 (procmime.c:1501)
==30919==    by 0x59B089: procmime_scan_file_with_offset (procmime.c:2174)
==30919== 
==30919== Invalid read of size 4
==30919==    at 0x1C6BCF32: PyObject_Free (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C65E5DA: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C65E89A: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C613316: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C6C2133: PyDict_SetItem (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C605D15: _PyModule_Clear (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C60603D: PyImport_Cleanup (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C58EF9A: Py_Finalize (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C32B277: plugin_done (python_plugin.c:717)
==30919==    by 0x662665: plugin_unload (plugin.c:545)
==30919==    by 0x662B4F: plugin_unload_all (plugin.c:639)
==30919==    by 0x4D987B: exit_claws (main.c:1668)
==30919==  Address 0x1cdbc020 is not stack'd, malloc'd or (recently) free'd
==30919== 
==30919== Invalid read of size 4
==30919==    at 0x1C6BCF32: PyObject_Free (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C6A37CA: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C65E64A: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C65E89A: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C613316: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C6C2133: PyDict_SetItem (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C605D15: _PyModule_Clear (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C60603D: PyImport_Cleanup (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C58EF9A: Py_Finalize (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C32B277: plugin_done (python_plugin.c:717)
==30919==    by 0x662665: plugin_unload (plugin.c:545)
==30919==    by 0x662B4F: plugin_unload_all (plugin.c:639)
==30919==  Address 0x1cdbd020 is 592 bytes inside a block of size 1,053 free'd
==30919==    at 0x4C2A739: free (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==30919==    by 0x1C65E5DA: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C65E89A: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C613316: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C6C2133: PyDict_SetItem (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C605D15: _PyModule_Clear (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C60603D: PyImport_Cleanup (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C58EF9A: Py_Finalize (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C32B277: plugin_done (python_plugin.c:717)
==30919==    by 0x662665: plugin_unload (plugin.c:545)
==30919==    by 0x662B4F: plugin_unload_all (plugin.c:639)
==30919==    by 0x4D987B: exit_claws (main.c:1668)
==30919== 
==30919== Invalid read of size 4
==30919==    at 0x1C6BCF32: PyObject_Free (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C65E5B0: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C65E89A: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C613316: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C6C2133: PyDict_SetItem (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C605D15: _PyModule_Clear (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C60603D: PyImport_Cleanup (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C58EF9A: Py_Finalize (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C32B277: plugin_done (python_plugin.c:717)
==30919==    by 0x662665: plugin_unload (plugin.c:545)
==30919==    by 0x662B4F: plugin_unload_all (plugin.c:639)
==30919==    by 0x4D987B: exit_claws (main.c:1668)
==30919==  Address 0x1cdca020 is not stack'd, malloc'd or (recently) free'd
==30919== 
==30919== Invalid read of size 4
==30919==    at 0x1C6BCF32: PyObject_Free (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C6A37CA: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C65E65A: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C65E89A: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C613316: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C6C2133: PyDict_SetItem (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C605DAF: _PyModule_Clear (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C60603D: PyImport_Cleanup (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C58EF9A: Py_Finalize (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C32B277: plugin_done (python_plugin.c:717)
==30919==    by 0x662665: plugin_unload (plugin.c:545)
==30919==    by 0x662B4F: plugin_unload_all (plugin.c:639)
==30919==  Address 0x1cdc0020 is 48 bytes inside a block of size 1,205 free'd
==30919==    at 0x4C2A739: free (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==30919==    by 0x1C65E5DA: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C65E89A: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C613316: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C6C2133: PyDict_SetItem (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C605DAF: _PyModule_Clear (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C60603D: PyImport_Cleanup (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C58EF9A: Py_Finalize (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C32B277: plugin_done (python_plugin.c:717)
==30919==    by 0x662665: plugin_unload (plugin.c:545)
==30919==    by 0x662B4F: plugin_unload_all (plugin.c:639)
==30919==    by 0x4D987B: exit_claws (main.c:1668)
==30919== 
==30919== Invalid read of size 4
==30919==    at 0x1C6BCF32: PyObject_Free (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C65E8AA: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C613316: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C6C2133: PyDict_SetItem (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C605DAF: _PyModule_Clear (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C60603D: PyImport_Cleanup (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C58EF9A: Py_Finalize (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C32B277: plugin_done (python_plugin.c:717)
==30919==    by 0x662665: plugin_unload (plugin.c:545)
==30919==    by 0x662B4F: plugin_unload_all (plugin.c:639)
==30919==    by 0x4D987B: exit_claws (main.c:1668)
==30919==    by 0x4D95FA: main (main.c:1574)
==30919==  Address 0x1cdc0020 is 48 bytes inside a block of size 1,205 free'd
==30919==    at 0x4C2A739: free (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==30919==    by 0x1C65E5DA: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C65E89A: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C613316: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C6C2133: PyDict_SetItem (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C605DAF: _PyModule_Clear (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C60603D: PyImport_Cleanup (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C58EF9A: Py_Finalize (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C32B277: plugin_done (python_plugin.c:717)
==30919==    by 0x662665: plugin_unload (plugin.c:545)
==30919==    by 0x662B4F: plugin_unload_all (plugin.c:639)
==30919==    by 0x4D987B: exit_claws (main.c:1668)
==30919== 
==30919== Invalid read of size 4
==30919==    at 0x1C6BCF32: PyObject_Free (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C613316: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C6C2133: PyDict_SetItem (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C605DAF: _PyModule_Clear (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C60603D: PyImport_Cleanup (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C58EF9A: Py_Finalize (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C32B277: plugin_done (python_plugin.c:717)
==30919==    by 0x662665: plugin_unload (plugin.c:545)
==30919==    by 0x662B4F: plugin_unload_all (plugin.c:639)
==30919==    by 0x4D987B: exit_claws (main.c:1668)
==30919==    by 0x4D95FA: main (main.c:1574)
==30919==  Address 0x1d03b020 is not stack'd, malloc'd or (recently) free'd
==30919== 
==30919== Invalid read of size 4
==30919==    at 0x1C6BCF32: PyObject_Free (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C6A367E: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C613316: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C6C2133: PyDict_SetItem (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C605DAF: _PyModule_Clear (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C60603D: PyImport_Cleanup (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C58EF9A: Py_Finalize (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C32B277: plugin_done (python_plugin.c:717)
==30919==    by 0x662665: plugin_unload (plugin.c:545)
==30919==    by 0x662B4F: plugin_unload_all (plugin.c:639)
==30919==    by 0x4D987B: exit_claws (main.c:1668)
==30919==    by 0x4D95FA: main (main.c:1574)
==30919==  Address 0x1ce75020 is not stack'd, malloc'd or (recently) free'd
==30919== 
==30919== Invalid read of size 4
==30919==    at 0x1C6BCF32: PyObject_Free (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C6A37B1: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C65E65A: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C65E89A: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C613316: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C6C2133: PyDict_SetItem (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C605D15: _PyModule_Clear (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C60603D: PyImport_Cleanup (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C58EF9A: Py_Finalize (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C32B277: plugin_done (python_plugin.c:717)
==30919==    by 0x662665: plugin_unload (plugin.c:545)
==30919==    by 0x662B4F: plugin_unload_all (plugin.c:639)
==30919==  Address 0x1cc85020 is not stack'd, malloc'd or (recently) free'd
==30919== 
==30919== Conditional jump or move depends on uninitialised value(s)
==30919==    at 0x1C6BCF3B: PyObject_Free (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C65E5B0: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C65E89A: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C613316: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C6C2133: PyDict_SetItem (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C605DAF: _PyModule_Clear (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C60603D: PyImport_Cleanup (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C58EF9A: Py_Finalize (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C32B277: plugin_done (python_plugin.c:717)
==30919==    by 0x662665: plugin_unload (plugin.c:545)
==30919==    by 0x662B4F: plugin_unload_all (plugin.c:639)
==30919==    by 0x4D987B: exit_claws (main.c:1668)
==30919== 
==30919== Use of uninitialised value of size 8
==30919==    at 0x1C6BCF53: PyObject_Free (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C65E5B0: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C65E89A: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C613316: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C6C2133: PyDict_SetItem (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C605DAF: _PyModule_Clear (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C60603D: PyImport_Cleanup (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C58EF9A: Py_Finalize (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C32B277: plugin_done (python_plugin.c:717)
==30919==    by 0x662665: plugin_unload (plugin.c:545)
==30919==    by 0x662B4F: plugin_unload_all (plugin.c:639)
==30919==    by 0x4D987B: exit_claws (main.c:1668)
==30919== 
==30919== Invalid read of size 4
==30919==    at 0x1C6BCF32: PyObject_Free (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C65E5DA: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C65E89A: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C6A367E: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C65E2AA: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C613316: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C6C2133: PyDict_SetItem (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C605DAF: _PyModule_Clear (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C60603D: PyImport_Cleanup (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C58EF9A: Py_Finalize (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C32B277: plugin_done (python_plugin.c:717)
==30919==    by 0x662665: plugin_unload (plugin.c:545)
==30919==  Address 0x1d17b020 is 48 bytes inside a block of size 72 free'd
==30919==    at 0x4C2A739: free (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==30919==    by 0xC7ACF82: asn1_delete_structure (in /usr/lib/x86_64-linux-gnu/libtasn1.so.3.1.16)
==30919==    by 0x6EF9BA3: gnutls_x509_crt_deinit (in /usr/lib/x86_64-linux-gnu/libgnutls.so.26.21.8)
==30919==    by 0x6EB1D99: gnutls_certificate_free_cas (in /usr/lib/x86_64-linux-gnu/libgnutls.so.26.21.8)
==30919==    by 0x6EB2B50: gnutls_certificate_free_credentials (in /usr/lib/x86_64-linux-gnu/libgnutls.so.26.21.8)
==30919==    by 0x66DD1E: ssl_done_socket (ssl.c:400)
==30919==    by 0x66D128: sock_close (socket.c:1630)
==30919==    by 0x66551C: session_close (session.c:362)
==30919==    by 0x664D43: session_disconnect (session.c:210)
==30919==    by 0x52BCD8: pop3_session_recv_msg (pop.c:1039)
==30919==    by 0x6667E9: session_read_msg_cb (session.c:605)
==30919==    by 0x66AD78: ssl_sock_dispatch (socket.c:505)
==30919== 
==30919== Invalid read of size 4
==30919==    at 0x1C6BCF32: PyObject_Free (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C6A37CA: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C65E63A: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C65E89A: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C6A367E: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C65E2AA: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C613316: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C6C2133: PyDict_SetItem (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C605DAF: _PyModule_Clear (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C60603D: PyImport_Cleanup (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C58EF9A: Py_Finalize (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C32B277: plugin_done (python_plugin.c:717)
==30919==  Address 0x1d17c020 is 288 bytes inside a block of size 997 free'd
==30919==    at 0x4C2A739: free (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==30919==    by 0x1C65E5DA: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C65E89A: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C6A367E: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C65E2AA: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C613316: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C6C2133: PyDict_SetItem (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C605DAF: _PyModule_Clear (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C60603D: PyImport_Cleanup (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C58EF9A: Py_Finalize (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C32B277: plugin_done (python_plugin.c:717)
==30919==    by 0x662665: plugin_unload (plugin.c:545)
==30919== 
==30919== Invalid read of size 4
==30919==    at 0x1C6BCF32: PyObject_Free (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C613316: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C6C2133: PyDict_SetItem (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C605D15: _PyModule_Clear (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C60603D: PyImport_Cleanup (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C58EF9A: Py_Finalize (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C32B277: plugin_done (python_plugin.c:717)
==30919==    by 0x662665: plugin_unload (plugin.c:545)
==30919==    by 0x662B4F: plugin_unload_all (plugin.c:639)
==30919==    by 0x4D987B: exit_claws (main.c:1668)
==30919==    by 0x4D95FA: main (main.c:1574)
==30919==  Address 0x1d212020 is not stack'd, malloc'd or (recently) free'd
==30919== 
==30919== Invalid read of size 4
==30919==    at 0x1C6BCF32: PyObject_Free (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C6A367E: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C65E2AA: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C613316: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C6C2133: PyDict_SetItem (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C605DAF: _PyModule_Clear (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C60603D: PyImport_Cleanup (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C58EF9A: Py_Finalize (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C32B277: plugin_done (python_plugin.c:717)
==30919==    by 0x662665: plugin_unload (plugin.c:545)
==30919==    by 0x662B4F: plugin_unload_all (plugin.c:639)
==30919==    by 0x4D987B: exit_claws (main.c:1668)
==30919==  Address 0x1cffe020 is not stack'd, malloc'd or (recently) free'd
==30919== 
==30919== Invalid read of size 4
==30919==    at 0x1C6BCF32: PyObject_Free (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C65E8AA: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C6A367E: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C65E2AA: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C613316: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C6C2133: PyDict_SetItem (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C605DAF: _PyModule_Clear (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C60603D: PyImport_Cleanup (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C58EF9A: Py_Finalize (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C32B277: plugin_done (python_plugin.c:717)
==30919==    by 0x662665: plugin_unload (plugin.c:545)
==30919==    by 0x662B4F: plugin_unload_all (plugin.c:639)
==30919==  Address 0x1cffe020 is not stack'd, malloc'd or (recently) free'd
==30919== 
==30919== 
==30919== More than 100 errors detected.  Subsequent errors
==30919== will still be recorded, but in less detail than before.
==30919== Invalid read of size 4
==30919==    at 0x1C6BCF32: PyObject_Free (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C6A36BC: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C65E2AA: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C613316: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C6C2133: PyDict_SetItem (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C605DAF: _PyModule_Clear (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C606211: PyImport_Cleanup (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C58EF9A: Py_Finalize (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C32B277: plugin_done (python_plugin.c:717)
==30919==    by 0x662665: plugin_unload (plugin.c:545)
==30919==    by 0x662B4F: plugin_unload_all (plugin.c:639)
==30919==    by 0x4D987B: exit_claws (main.c:1668)
==30919==  Address 0x1d281020 is 0 bytes after a block of size 3,072 alloc'd
==30919==    at 0x4C2B3F8: malloc (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==30919==    by 0x1C6C1DB7: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C5806D7: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C584C37: PyEval_EvalCodeEx (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C584D71: PyEval_EvalCode (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C5DBB7C: PyImport_ExecCodeModuleEx (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C680B77: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C6812EB: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C638689: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C6817FC: PyImport_ImportModuleLevel (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C5D703E: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C6541FD: PyObject_Call (in /usr/lib/libpython2.7.so.1.0)
==30919== 
==30919== Conditional jump or move depends on uninitialised value(s)
==30919==    at 0x1C6BCF3B: PyObject_Free (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C6A36BC: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C65E441: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C613316: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C6C2133: PyDict_SetItem (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C605DAF: _PyModule_Clear (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C606211: PyImport_Cleanup (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C58EF9A: Py_Finalize (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C32B277: plugin_done (python_plugin.c:717)
==30919==    by 0x662665: plugin_unload (plugin.c:545)
==30919==    by 0x662B4F: plugin_unload_all (plugin.c:639)
==30919==    by 0x4D987B: exit_claws (main.c:1668)
==30919== 
==30919== Use of uninitialised value of size 8
==30919==    at 0x1C6BCF53: PyObject_Free (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C6A36BC: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C65E441: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C613316: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C6C2133: PyDict_SetItem (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C605DAF: _PyModule_Clear (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C606211: PyImport_Cleanup (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C58EF9A: Py_Finalize (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C32B277: plugin_done (python_plugin.c:717)
==30919==    by 0x662665: plugin_unload (plugin.c:545)
==30919==    by 0x662B4F: plugin_unload_all (plugin.c:639)
==30919==    by 0x4D987B: exit_claws (main.c:1668)
==30919== 
==30919== Conditional jump or move depends on uninitialised value(s)
==30919==    at 0x1C6BCF3B: PyObject_Free (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C6A367E: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C6A367E: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C6147D9: _PyImport_Fini (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C58EFA4: Py_Finalize (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C32B277: plugin_done (python_plugin.c:717)
==30919==    by 0x662665: plugin_unload (plugin.c:545)
==30919==    by 0x662B4F: plugin_unload_all (plugin.c:639)
==30919==    by 0x4D987B: exit_claws (main.c:1668)
==30919==    by 0x4D95FA: main (main.c:1574)
==30919== 
==30919== Use of uninitialised value of size 8
==30919==    at 0x1C6BCF53: PyObject_Free (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C6A367E: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C6A367E: ??? (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C6147D9: _PyImport_Fini (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C58EFA4: Py_Finalize (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C32B277: plugin_done (python_plugin.c:717)
==30919==    by 0x662665: plugin_unload (plugin.c:545)
==30919==    by 0x662B4F: plugin_unload_all (plugin.c:639)
==30919==    by 0x4D987B: exit_claws (main.c:1668)
==30919==    by 0x4D95FA: main (main.c:1574)
==30919== 
==30919== Invalid read of size 4
==30919==    at 0x1C6BCF32: PyObject_Free (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C69613B: PyDict_Fini (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C58F009: Py_Finalize (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C32B277: plugin_done (python_plugin.c:717)
==30919==    by 0x662665: plugin_unload (plugin.c:545)
==30919==    by 0x662B4F: plugin_unload_all (plugin.c:639)
==30919==    by 0x4D987B: exit_claws (main.c:1668)
==30919==    by 0x4D95FA: main (main.c:1574)
==30919==  Address 0x1d27c020 is not stack'd, malloc'd or (recently) free'd
==30919== 
==30919== Conditional jump or move depends on uninitialised value(s)
==30919==    at 0x1C6BCF3B: PyObject_Free (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C69613B: PyDict_Fini (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C58F009: Py_Finalize (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C32B277: plugin_done (python_plugin.c:717)
==30919==    by 0x662665: plugin_unload (plugin.c:545)
==30919==    by 0x662B4F: plugin_unload_all (plugin.c:639)
==30919==    by 0x4D987B: exit_claws (main.c:1668)
==30919==    by 0x4D95FA: main (main.c:1574)
==30919== 
==30919== Use of uninitialised value of size 8
==30919==    at 0x1C6BCF53: PyObject_Free (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C69613B: PyDict_Fini (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C58F009: Py_Finalize (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C32B277: plugin_done (python_plugin.c:717)
==30919==    by 0x662665: plugin_unload (plugin.c:545)
==30919==    by 0x662B4F: plugin_unload_all (plugin.c:639)
==30919==    by 0x4D987B: exit_claws (main.c:1668)
==30919==    by 0x4D95FA: main (main.c:1574)
==30919== 
==30919== Invalid read of size 4
==30919==    at 0x1C6BCF32: PyObject_Free (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C695C95: PyGrammar_RemoveAccelerators (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C58F01A: Py_Finalize (in /usr/lib/libpython2.7.so.1.0)
==30919==    by 0x1C32B277: plugin_done (python_plugin.c:717)
==30919==    by 0x662665: plugin_unload (plugin.c:545)
==30919==    by 0x662B4F: plugin_unload_all (plugin.c:639)
==30919==    by 0x4D987B: exit_claws (main.c:1668)
==30919==    by 0x4D95FA: main (main.c:1574)
==30919==  Address 0x1ceb0020 is not stack'd, malloc'd or (recently) free'd
==30919== 
==30919== 
==30919== HEAP SUMMARY:
==30919==     in use at exit: 16,207,648 bytes in 81,753 blocks
==30919==   total heap usage: 5,025,627 allocs, 4,943,874 frees, 368,742,464 bytes allocated
==30919== 
==30919== LEAK SUMMARY:
==30919==    definitely lost: 49,337 bytes in 140 blocks
==30919==    indirectly lost: 43,152 bytes in 1,345 blocks
==30919==      possibly lost: 8,769,791 bytes in 55,021 blocks
==30919==    still reachable: 7,345,368 bytes in 25,247 blocks
==30919==         suppressed: 0 bytes in 0 blocks
==30919== Rerun with --leak-check=full to see details of leaked memory
==30919== 
==30919== For counts of detected and suppressed errors, rerun with: -v
==30919== Use --track-origins=yes to see where uninitialised values come from
==30919== ERROR SUMMARY: 2604 errors from 109 contexts (suppressed: 2 from 2)
